<!-- OPEN ACCESS Freely available online -->

<!-- PLOS COMPUTATIONAL BIOLOGY -->

# Mechanical Strength of 17 134 Model Proteins and Cysteine Slipknots

**Mateusz Sikora¹⁹, Joanna I. Sułkowska¹,2⁹, Marek Cieplak¹*⁹**

1 Institute of Physics, Polish Academy of Sciences, Warsaw, Poland, 2 Center for Theoretical Biological Physics, University of California, San Diego, California, USA

# Abstract

A new theoretical survey of proteins' resistance to constant speed stretching is performed for a set of 17 134 proteins as described by a structure-based model. The proteins selected have no gaps in their structure determination and consist of no more than 250 amino acids. Our previous studies have dealt with 7510 proteins of no more than 150 amino acids. The proteins are ranked according to the strength of the resistance. Most of the predicted top-strength proteins have not yet been studied experimentally. Architectures and folds which are likely to yield large forces are identified. New types of potent force clamps are discovered. They involve disulphide bridges and, in particular, cysteine slipknots. An effective energy parameter of the model is estimated by comparing the theoretical data on characteristic forces to the corresponding experimental vvalues combined with an extrapolation of the theoretical data to the experimental pulling speeds. These studies provide guidance for future experiments on single molecule manipulation and should lead to selection of proteins for applications. A new class of proteins, involving cystein slipknots, is identified as one that is expected to lead to the strongest force clamps known. This class is characterized through molecular dynamics simulations.

**Citation**: Sikora M, Sulkowska JI, Cieplak M (2009) Mechanical Strength of 17 134 Model Proteins and Cysteine Slipknots. PLoS Comput Biol 5(10): e1000547. doi:10.1371/journal.pcbi.1000547

**Editor**:Angel Garcia, Rensselaer Polytechnic Institute, United States of America

**Received** May 19, 2009; **Accepted** September 28, 2009; **Published** October 30,2009

**Copyright**: © 2009 Sikora et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.

**Funding:** The research leading to these results has received fundingfrom the European Community's Seventh Framework Programme (FP7/2007-2013) under **grant** agreement 213382 (FUNMOL). This work has also been supported by the grant N N202 0852 33 from the Ministry of Science and Higher Education in Poland and the Center for Theoretical Biological Physics at the UCSD sponsored by the NSF (Grant PHY-0822283). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.

**Competing** **Interests**: The authors have declared that no competing interests exist.

* E-mail:mc@ifpan.edu.pl

All authors contributed equally to this work.

# Introduction

Atomic force microscopy, optical tweezers, and other tools of nanotechnology have enabled induction and monitoring of large conformational changes in biomolecules. Such studies are performed to assess structure of the biomolecules, their elastic properties, and ability to act as nanomachines in a cell. Stretching studies of proteins [1] are of a particular current interest and they have been performed for under a hundred of systems. Interpre-tation of some of these experiments has been helped by all-atom simulations, such as reported in refs. [2,3]. They are limited by of order 100 ns time scales and thus require using unrealistically large constant pulling speeds. However, they often elucidate the nature of the force clamp - the region responsible for the largest force of resistance to pulling,FmaxAll of the experimental and all-atom simulational studies address merely a tiny fraction of proteins that are stored in the Protein Data Bank (PDB) [4]. Thus it appears worthwhile to consider a large set of proteins and determine theirFmaxwithin an approximate model that allows for fast and yet reasonably accurate calculations. Structure-based models of proteins, as pioneered by Go and his collaborators [5] and used in several implementations [6-13],seem to be suited to this task especially well since they are defined in terms of the native structures away from which stretching is imposed.

There are many ways, all phenomenological, to construct a structure-based model of a protein. 504 of possible variants are enumerated and 62 are studied in details in ref. [14]. The variants differ by the choice of effective potentials, nature of the local backbone stiffness,energy-related parameters, and of the coarse-grained degrees of freedom. The most crucial choice relates to making a decision about which interactions between amino acids count as native contacts. Comparing Fmax to the corresponding experimental values in 36 available cases selects several optimal models [14]. Among them, there is one which is very simple and which describes a protein in terms of itsCx atoms,as labeled by the sequential index i. This model is denoted by LJ3={6-12,C,M3,.E0} which stands for,respectively,the Lennard-Jones native contact potentials, local backbone stiffness represented by harmonic terms that favor the native values of local chiralities,the contact map in which there are noi,i+2contacts, and the amplitude of the Lennard-Jones potential,ϵ,, is uniform. The contact map is determined by assigning the van der Waals spheres to the heavy atoms (enlarged by a factor to account for attraction) and by checking whether spheres belonging to different amino acids overlap in the native state [15,16].If they do, a contact is declared as native. Non-native contacts are considered repulsive. Application of this criterion frequently selects thei,i+2contacts as native. If the contact map includes these contacts the resulting model will be denoted here as LJ2. On average, it performs worse than LJ3because thei,i+2contacts usually correspond to the weak van der WaaIs couplings as can be demonstrated in a sample of proteins by using a software [17] which analyses atomic configurations from the


![](https://web-api.textin.com/ocr_image/external/d06de67514d05b7a.jpg)

<!-- PLoS Computational Biology | www.ploscompbiol.org 1 October 2009 | Volume **5** | Issue 10 | e1000547 -->

<!-- Mechanical Strength of 17 134 Model Proteins -->

# Author Summary

<table border="1" ><tr>
<td>The advances in nanotechnology have allowed for manipulation of single biomolecules and determination of their elastic properties. Titin was among the first proteins studied in this way. Its unravelling by stretching requires a 204 pN force. The resistance to stretching comes mostly from a localized region known as a force clamp. In titin, the force clamp is simple as it is formed by two parallelβ-strands that are sheared on pulling. Studies of a set of under a huundred proteins accomplished in the last decade have revealed a variety of the force clamps that lead to forces ranging from under 20 pN to about 500pN. This set comprises only a tiny fraction of proteins known. Thus one needs guidance as to what proteins should be considered for specific mechanical properties. Such a guidance is provided here through simulations within simplified coarse-grained models on 17 134 proteins that are stretched at constant speed. We correlate their unravelling forces with two structure classification schemes. We identify proteins with large resistance to unravelling and characterize their force clamps. Quite a few top strength proteins owe their sturdiness to a new type of the force clamp: the cystein slipknot in which the force peak is due to dragging of a piece of the backbone through a closed ring formed by two other pieces of the backbone and two connecting disulphide bonds.</td>
</tr></table>

chemical perspective on molecular bonds. Thus thei,i+2couplings should better be removed from the contact map (in most cases).

The survey to determineFmax in 7510 model proteins with the number of amino acids, N,not exceeding 150 and 239 longer proteins (with N up to 851) has been accomplished twice. First within the LJ2 model [18] and soon afterwords within the LJ3model [19]. The first survey also comes with many details of the methodology whereas the second just presents the outcomes.The two surveys are compared in more details in refs. [14,20].The results differ, particularly when it comes to ranking of the proteins according to the value ofFmax,but they mutually provide the error bars on the findings. They both agree, however, on predicting that there are many proteins whose strength should be considerably larger than the frequently studied benchmark- the sarcomere protein titin(Fmax.of order 204 pN [21,22]). Near the top of the list, there is the scaffoldin protein c7A (the PDB code laoh) which has been recently measured to have.Fmaxof about 480pN [23]. Other findings include establishing correlations with the CATH hierar-chical classification scheme [24,25], such as that there are no strong α proteins, and identification of several types of the force clamps. The large forces most commonly originate in parallel β-strands that are sheared [26]. However, there are also clamps with antiparallelβ- strands, unstructured strands,and other kinds.

The two surveys have been based on the structure download made on July 26, 2005 when the PDB comprised 29 385 entries. Many of them correspond to nucleic acids, complexes with nucleic acids and with other proteins, carbohydrates, or come with incomplete files and hencethe much smaller number of proteins that could be used in the molecular dynamics studies. Here, we present results of still another survey which is based on a download of December 18, 2008 which contains 54 807 structure files and leads to 17 134 acceptable structures with N not exceeding 250(instead of 150). These structures are then analyzed through simulations based on the LJ3 model. The numerical code has been improved to allow for acceleration of calculations by a factor of 2.

The 190 structures (or 1.1% of all structure considered) with the top values ofFmax in units ofϵ/AAre shown in Table 1 (the first 81entries for which.Fmax≥3.9ϵ/AA)and Table S1 of the SI (proteins ranked 82 through 190), together with the values of titin (1tit) and ubiquitin (lubq) to provide a scale. As argued in the Materials and Methods section section, the unit of force,ϵ/AA is now estimated to be of order 110pN. Al1 of the corresponding proteins are predicted to be much stronger than titin and none but two of them (laho, lglk [23]) have been studied experimentally yet. In addition to the types of force clamps identified before, we have discovered two new mechanisms of sturdiness. One of them involves a cysteine slipknot (CSK) and is found to be operational in all of the 13 top strength proteins. In this motif, a slip-loop is pulled out of a cysteine knot-loop. Another involves dragging of a single fragment of the main chain across a cysteine knot-loop. The two mechanisms are similar in spirit since both involve dragging of the backbone. However, in the CSK case, two fragments of the backbone are participating.

We make a more systematic identification of the CATH-classified architectures that are linked to mechanical strength and then analyze correlations of the data to the SCOP-based grouping (version 1.73) [27-29].The previous surveys did not relate to the SCOP scheme.

We identify the CATH-based architectures and SCOP-based folds that are associated with the occurrence of a strong resistance to pulling. A general observation, however, is that each such group of structures may also include examples of proteins that unravel easily. The dynamics of a protein are very sensitive to mechanical details that are largely captured by the contact map and not just by the appearance of a structure. On the other hand, if one were to look for mechanically strong proteins then the architectures and folds identified by us should provide a good starting point. We also study the dependence ofFmax on the pulling velocity and characterize the dependence on N through distributions of the forces.

The current third survey has been performed within the same LJ3 model as the second survey [19]. However, we reuse and extend it here because the editors of Biophysical Journal retracted the second survey [30]. All of the values ofFmaxare deposited at the website www.ifpan.edu.pl/BSDB (Biomolecule Stretching Data-base) and can by accessed by through the PDB structure code.

# Results/Discussion

# Distribution of Forces

The distribution of all values ofFmaxfor the full set of proteins is shown in Figure 1. Despite the larger limit on N now allowed, the distribution is rather similar to that obtained in ref.[19]for the smaller number of proteins (and with the smaller sizes). The similarity is primarily due to the fact that the size related effects, discussed below, are countered by new types of proteins that are now incorporated into the survey. The distribution is peaked aroundFmaxof1.2ϵ/AAwhich constitutes about 60% of the strength associated with titin. The distribution is non-Gaussian: it has a zero-force peak and a long force tail. The zero-force peak arises in some proteins with the covalent disulphide bonds. In the model, such bonds are represented by strong harmonic bonds. Stretching of such a protein may not result in any force peak before a disulphide bond gets stretched indefinitely and hence Fmaxis considered to be vanishing then. The tail, on the other hand, corresponds to the strong proteins. The top strongest 1.1% of all proteins are listed in Tables 1 (in the main text) and S1 (in the SI).

The insets of Figure 1 show similar distributions for proteins belonging to the particular CATH-based classes. There are four such classes: α, ,βα-β and proteins with no apparent secondary structures. It is seen that none of the 3240 α proteins exceeds the peak force obtained for titin within our model. This observation is in agreement with experiments on severalαproteins that are listed in


![](https://web-api.textin.com/ocr_image/external/c4dc9087a6650740.jpg)

<!-- PLoS Computational Biology | www.ploscompbiol.org 2 October 2009 | Volume 5 | Issue 10 | e1000547 -->

<!-- Mechanical Strength of 17 134 Model Proteins -->

**Table** **1.** The predicted list of the strongest proteins.

<table border="1" ><tr>
<td>n</td>
<td>PDBid</td>
<td>N</td>
<td>Fmax[ϵ/AA]</td>
<td>Lmax[AA]</td>
<td>λ</td>
<td>CATH</td>
<td>SCOP</td>
</tr><tr>
<td>1</td>
<td>1bmp</td>
<td>104</td>
<td>10.2</td>
<td>23.2</td>
<td>0.01</td>
<td>2.10.90.10</td>
<td>g.17.1.2</td>
</tr><tr>
<td>2</td>
<td>1qty</td>
<td>95</td>
<td>8.9</td>
<td>72.1</td>
<td>0.11</td>
<td>2.10.90.10</td>
<td>b.1.1.4</td>
</tr><tr>
<td>3</td>
<td>2bhk</td>
<td>119</td>
<td>7.3</td>
<td>26.5</td>
<td>0.67</td>
<td></td>
<td></td>
</tr><tr>
<td>4</td>
<td>1lxi</td>
<td>104</td>
<td>7.3</td>
<td>22.5</td>
<td>0.01</td>
<td></td>
<td>g.17.1.2</td>
</tr><tr>
<td>5</td>
<td>1cz8</td>
<td>107</td>
<td>6.4</td>
<td>76.5</td>
<td>0.13</td>
<td>2.10.90.10</td>
<td>b.1.1.1</td>
</tr><tr>
<td>6</td>
<td>2gh0</td>
<td>219</td>
<td>5.8</td>
<td>25.9</td>
<td>0.06</td>
<td></td>
<td></td>
</tr><tr>
<td>7</td>
<td>1wq9</td>
<td>100</td>
<td>5.5</td>
<td>72.0</td>
<td>0.10</td>
<td>2.10.90.10</td>
<td>g.17.1.1</td>
</tr><tr>
<td>8</td>
<td>1flt</td>
<td>107</td>
<td>5.5</td>
<td>75.6</td>
<td>0.12</td>
<td>2.10.90.10</td>
<td>b.1.1.4</td>
</tr><tr>
<td>9</td>
<td>1fzv</td>
<td>117</td>
<td>5.4</td>
<td>90.4</td>
<td>0.12</td>
<td>2.10.90.10</td>
<td>g.17.1.1</td>
</tr><tr>
<td>10</td>
<td>2gyz</td>
<td>100</td>
<td>5.4</td>
<td>14.4</td>
<td>0.01</td>
<td></td>
<td></td>
</tr><tr>
<td>11</td>
<td>1rew</td>
<td>103</td>
<td>5.3</td>
<td>21.7</td>
<td>0.01</td>
<td>2.10.90.10</td>
<td>g.7.1.3</td>
</tr><tr>
<td>12</td>
<td>1m4u</td>
<td>139</td>
<td>5.3</td>
<td>52.1</td>
<td>0.07</td>
<td>2.10.90.10</td>
<td>g.17.1.2</td>
</tr><tr>
<td>13</td>
<td>1vpf</td>
<td>94</td>
<td>5.3</td>
<td>68.1</td>
<td>0.11</td>
<td>2.10.90.10</td>
<td>g.17.1.1</td>
</tr><tr>
<td>14</td>
<td>1c4p</td>
<td>137</td>
<td>5.1</td>
<td>106.0</td>
<td>0.12</td>
<td>3.10.20.180</td>
<td>d.15.5.1</td>
</tr><tr>
<td>15</td>
<td>1qqr</td>
<td>138</td>
<td>5.0</td>
<td>110.3</td>
<td>0.12</td>
<td>3.10.20.180</td>
<td>d.15.5.1</td>
</tr><tr>
<td>16</td>
<td>3bmp</td>
<td>114</td>
<td>5.0</td>
<td>33.0</td>
<td>0.03</td>
<td>2.10.90.10</td>
<td>g.17.1.2</td>
</tr><tr>
<td>17</td>
<td>1j8s</td>
<td>193</td>
<td>4.9</td>
<td>77.9</td>
<td>0.03</td>
<td>2.60.40.1370</td>
<td>b.2.3.3</td>
</tr><tr>
<td>18</td>
<td>1wq8</td>
<td>96</td>
<td>4.9</td>
<td>82.6</td>
<td>0.11</td>
<td>2.10.90.10</td>
<td>g.17.1.1</td>
</tr><tr>
<td>19</td>
<td>1j8r</td>
<td>193</td>
<td>4.8</td>
<td>77.7</td>
<td>0.03</td>
<td>2.60.40.1370</td>
<td>b.2.3.3</td>
</tr><tr>
<td>20</td>
<td>1f3y</td>
<td>165</td>
<td>4.8</td>
<td>284.7</td>
<td>0.43</td>
<td>3.90.79.10</td>
<td>d.113.1.1</td>
</tr><tr>
<td>21</td>
<td>2vpf</td>
<td>109</td>
<td>4.7</td>
<td>79.3</td>
<td>0.11</td>
<td>2.10.90.10</td>
<td>g.17.1.1</td>
</tr><tr>
<td>22</td>
<td>2h64</td>
<td>105</td>
<td>4.6</td>
<td>29.4</td>
<td>0.03</td>
<td></td>
<td>g.7.1.3</td>
</tr><tr>
<td>23</td>
<td>1kdm</td>
<td>177</td>
<td>4.6</td>
<td>309.4</td>
<td>0.45</td>
<td>2.60.120.200</td>
<td>b.29.1.4</td>
</tr><tr>
<td>24</td>
<td>1q56</td>
<td>195</td>
<td>4.5</td>
<td>473.2</td>
<td>0.62</td>
<td>2.60.120.200</td>
<td>b.29.1.4</td>
</tr><tr>
<td>25</td>
<td>1rv6</td>
<td>94</td>
<td>4.5</td>
<td>67.7</td>
<td>0.11</td>
<td>2.10.90.10</td>
<td>b.1.1.4</td>
</tr><tr>
<td>26</td>
<td>1waq</td>
<td>104</td>
<td>4.5</td>
<td>20.1</td>
<td>0.01</td>
<td></td>
<td></td>
</tr><tr>
<td>27</td>
<td>1reu</td>
<td>103</td>
<td>4.5</td>
<td>20.4</td>
<td>0.01</td>
<td>2.10.90.10</td>
<td>g.17.1.2</td>
</tr><tr>
<td>28</td>
<td>1tgj</td>
<td>112</td>
<td>4.4</td>
<td>45.9</td>
<td>0.07</td>
<td>2.10.90.10</td>
<td>g.17.1.2</td>
</tr><tr>
<td>29</td>
<td>2pbt</td>
<td>133</td>
<td>4.4</td>
<td>219.9</td>
<td>0.39</td>
<td></td>
<td></td>
</tr><tr>
<td>30</td>
<td>2h62</td>
<td>104</td>
<td>4.4</td>
<td>24.3</td>
<td>0.02</td>
<td></td>
<td>g.7.1.3</td>
</tr><tr>
<td>31</td>
<td>1tgk</td>
<td>112</td>
<td>4.4</td>
<td>44.6</td>
<td>0.07</td>
<td>2.10.90.10</td>
<td>g.17.1.2</td>
</tr><tr>
<td>32</td>
<td>2fzl</td>
<td>197</td>
<td>4.4</td>
<td>49.7</td>
<td>0.02</td>
<td></td>
<td>c.37.1.19</td>
</tr><tr>
<td>33</td>
<td>1qu0</td>
<td>181</td>
<td>4.3</td>
<td>156.9</td>
<td>0.22</td>
<td>2.60.120.200</td>
<td>b.29.1.4</td>
</tr><tr>
<td>34</td>
<td>1f5f</td>
<td>172</td>
<td>4.3</td>
<td>186.2</td>
<td>0.28</td>
<td>2.60.120.200</td>
<td>b.29.1.4</td>
</tr><tr>
<td>35</td>
<td>1dzk</td>
<td>148</td>
<td>4.3</td>
<td>110.3</td>
<td>0.16</td>
<td>2.40.128.20</td>
<td>b.60.1.1</td>
</tr><tr>
<td>36</td>
<td>1aoh</td>
<td>147</td>
<td>4.3</td>
<td>77.1</td>
<td>0.01</td>
<td>2.60.40.680</td>
<td>b.2.2.2</td>
</tr><tr>
<td>37</td>
<td>1vsc</td>
<td>196</td>
<td>4.3</td>
<td>238.3</td>
<td>0.24</td>
<td>2.60.40.10</td>
<td>b.1.1.3</td>
</tr><tr>
<td>38</td>
<td>2c7w</td>
<td>96</td>
<td>4.2</td>
<td>184.2</td>
<td>0.45</td>
<td>2.10.90.10</td>
<td></td>
</tr><tr>
<td>39</td>
<td>2gyr</td>
<td>97</td>
<td>4.2</td>
<td>27.1</td>
<td>0.05</td>
<td>2.10.90.10</td>
<td></td>
</tr><tr>
<td>40</td>
<td>1dzj</td>
<td>148</td>
<td>4.2</td>
<td>111.0</td>
<td>0.16</td>
<td>2.40.128.20</td>
<td>b.60.1.1</td>
</tr><tr>
<td>41</td>
<td>2sak</td>
<td>121</td>
<td>4.2</td>
<td>76.0</td>
<td>0.10</td>
<td>3.10.20.130</td>
<td>d.15.5.1</td>
</tr><tr>
<td>42</td>
<td>2bzm</td>
<td>129</td>
<td>4.2</td>
<td>124.3</td>
<td>0.24</td>
<td></td>
<td></td>
</tr><tr>
<td>43</td>
<td>2pq1</td>
<td>134</td>
<td>4.1</td>
<td>222.6</td>
<td>0.39</td>
<td></td>
<td></td>
</tr><tr>
<td>44</td>
<td>1nwv</td>
<td>129</td>
<td>4.1</td>
<td>129.8</td>
<td>0.13</td>
<td>2.10.70.10</td>
<td>g.18.1.1</td>
</tr><tr>
<td>45</td>
<td>1e5g</td>
<td>120</td>
<td>4.1</td>
<td>133.1</td>
<td>0.17</td>
<td>2.10.70.10</td>
<td>g.18.1.1</td>
</tr><tr>
<td>46</td>
<td>2ick</td>
<td>220</td>
<td>4.1</td>
<td>462.5</td>
<td>0.54</td>
<td></td>
<td></td>
</tr><tr>
<td>47</td>
<td>1gvl</td>
<td>223</td>
<td>4.1</td>
<td>114.9</td>
<td>0.09</td>
<td>2.40.10.10</td>
<td>b.47.1.2</td>
</tr><tr>
<td>48</td>
<td>1tgs</td>
<td>225</td>
<td>4.1</td>
<td>122.3</td>
<td>0.10</td>
<td>2.40.10.10</td>
<td>b.47.1.2</td>
</tr><tr>
<td>49</td>
<td>1u20</td>
<td>196</td>
<td>4.0</td>
<td>408.5</td>
<td>0.53</td>
<td></td>
<td>d.113.1.1</td>
</tr><tr>
<td>50</td>
<td>1cui</td>
<td>197</td>
<td>4.0</td>
<td>422.8</td>
<td>0.55</td>
<td>3.40.50.1820</td>
<td>c.69.1.30</td>
</tr></table>

<!-- PLoS Computational Biology | www.ploscompbiol.org 3 October 2009 | Volume 5 | Issue 10 | e1000547 -->

<!--  -->

<!-- Mechanical Strength of 17 134 Model Proteins -->

**Table** 1.Cont.

<table border="1" ><tr>
<td>n</td>
<td>PDBid</td>
<td>N</td>
<td>Fmax[ϵ/AA]</td>
<td>Lmax[AA]</td>
<td>2</td>
<td>CATH</td>
<td>SCOP</td>
</tr><tr>
<td>51</td>
<td>1ffd</td>
<td>197</td>
<td>4.0</td>
<td>423.0</td>
<td>0.55</td>
<td>3.40.50.1820</td>
<td>c.69.1.30</td>
</tr><tr>
<td>52</td>
<td>1kdk</td>
<td>177</td>
<td>4.0</td>
<td>357.2</td>
<td>0.53</td>
<td>2.60.120.200</td>
<td>b.29.1.4</td>
</tr><tr>
<td>53</td>
<td>2icj</td>
<td>219</td>
<td>4.0</td>
<td>455.9</td>
<td>0.53</td>
<td></td>
<td></td>
</tr><tr>
<td>54</td>
<td>3dd5</td>
<td>194</td>
<td>4.0</td>
<td>403.3</td>
<td>0.53</td>
<td></td>
<td></td>
</tr><tr>
<td>55</td>
<td>1cug</td>
<td>197</td>
<td>4.0</td>
<td>422.6</td>
<td>0.55</td>
<td>3.40.50.1820</td>
<td>c.69.1.30</td>
</tr><tr>
<td>56</td>
<td>1b0o</td>
<td>161</td>
<td>4.0</td>
<td>237.3</td>
<td>0.36</td>
<td>2.40.128.20</td>
<td>b.60.1.1</td>
</tr><tr>
<td>57</td>
<td>1xza</td>
<td>197</td>
<td>4.0</td>
<td>422.9</td>
<td>0.55</td>
<td>3.40.50.1820</td>
<td>c.69.1.30</td>
</tr><tr>
<td>58</td>
<td>1vcd</td>
<td>126</td>
<td>4.0</td>
<td>199.7</td>
<td>0.37</td>
<td></td>
<td>d.113.1.1</td>
</tr><tr>
<td>59</td>
<td>1cuw</td>
<td>197</td>
<td>4.0</td>
<td>422.9</td>
<td>0.55</td>
<td>3.40.50.1820</td>
<td>c.69.1.30</td>
</tr><tr>
<td>60</td>
<td>1xzi</td>
<td>197</td>
<td>4.0</td>
<td>422.9</td>
<td>0.55</td>
<td>3.40.50.1820</td>
<td>c.69.1.30</td>
</tr><tr>
<td>61</td>
<td>1cus</td>
<td>197</td>
<td>4.0</td>
<td>423.3</td>
<td>0.55</td>
<td>3.40.50.1820</td>
<td>c.69.1.30</td>
</tr><tr>
<td>62</td>
<td>1cuf</td>
<td>197</td>
<td>4.0</td>
<td>423.1</td>
<td>0.55</td>
<td>3.40.50.1820</td>
<td>c.69.1.30</td>
</tr><tr>
<td>63</td>
<td>2a7h</td>
<td>223</td>
<td>4.0</td>
<td>114.7</td>
<td>0.10</td>
<td>2.40.10.10</td>
<td>b.47.1.2</td>
</tr><tr>
<td>64</td>
<td>1cq3</td>
<td>224</td>
<td>4.0</td>
<td>128.0</td>
<td>0.12</td>
<td>2.60.240.10</td>
<td>b.27.1.1</td>
</tr><tr>
<td>65</td>
<td>1ffc</td>
<td>197</td>
<td>3.9</td>
<td>421.6</td>
<td>0.55</td>
<td>3.40.50.1820</td>
<td>c.69.1.30</td>
</tr><tr>
<td>66</td>
<td>1vc9</td>
<td>126</td>
<td>3.9</td>
<td>199.1</td>
<td>0.37</td>
<td></td>
<td>d.113.1.1</td>
</tr><tr>
<td>67</td>
<td>1cua</td>
<td>197</td>
<td>3.9</td>
<td>423.0</td>
<td>0.55</td>
<td>3.40.50.1820</td>
<td>c.69.1.30</td>
</tr><tr>
<td>68</td>
<td>1xzl</td>
<td>197</td>
<td>3.9</td>
<td>423.1</td>
<td>0.55</td>
<td>3.40.50.1820</td>
<td>c.69.1.30</td>
</tr><tr>
<td>69</td>
<td>2faw</td>
<td>250</td>
<td>3.9</td>
<td>250.8</td>
<td>0.25</td>
<td></td>
<td></td>
</tr><tr>
<td>70</td>
<td>2vn5</td>
<td>142</td>
<td>3.9</td>
<td>49.2</td>
<td>0.02</td>
<td></td>
<td></td>
</tr><tr>
<td>71</td>
<td>1cux</td>
<td>197</td>
<td>3.9</td>
<td>421.5</td>
<td>0.55</td>
<td>3.40.50.1820</td>
<td>c.69.1.30</td>
</tr><tr>
<td>72</td>
<td>1cuh</td>
<td>197</td>
<td>3.9</td>
<td>421.6</td>
<td>0.55</td>
<td>3.40.50.1820</td>
<td>c.69.1.30</td>
</tr><tr>
<td>73</td>
<td>2dsd</td>
<td>195</td>
<td>3.9</td>
<td>429.7</td>
<td>0.56</td>
<td></td>
<td></td>
</tr><tr>
<td>74</td>
<td>2f3c</td>
<td>221</td>
<td>3.9</td>
<td>113.5</td>
<td>0.10</td>
<td>2.40.10.10</td>
<td>b.47.1.2</td>
</tr><tr>
<td>75</td>
<td>1xzj</td>
<td>197</td>
<td>3.9</td>
<td>421.8</td>
<td>0.55</td>
<td>3.40.50.1820</td>
<td>c.69.1.30</td>
</tr><tr>
<td>76</td>
<td>1xzf</td>
<td>197</td>
<td>3.9</td>
<td>421.0</td>
<td>0.55</td>
<td>3.40.50.1820</td>
<td>c.69.1.30</td>
</tr><tr>
<td>77</td>
<td>2g7i</td>
<td>124</td>
<td>3.9</td>
<td>106.6</td>
<td>0.10</td>
<td></td>
<td></td>
</tr><tr>
<td>78</td>
<td>1g1k</td>
<td>143</td>
<td>3.9</td>
<td>52.0</td>
<td>0.02</td>
<td>2.60.40.680</td>
<td>b.2.2.2</td>
</tr><tr>
<td>79</td>
<td>1cuc</td>
<td>197</td>
<td>3.9</td>
<td>421.3</td>
<td>0.55</td>
<td>3.40.50.1820</td>
<td>c.69.1.30</td>
</tr><tr>
<td>80</td>
<td>1xzk</td>
<td>197</td>
<td>3.9</td>
<td>422.5</td>
<td>0.55</td>
<td>3.40.50.1820</td>
<td>c.69.1.30</td>
</tr><tr>
<td>81</td>
<td>1i04</td>
<td>159</td>
<td>3.9</td>
<td>231.7</td>
<td>0.34</td>
<td>2.40.128.20</td>
<td>b.60.1.1</td>
</tr><tr>
<td>3144</td>
<td>1ubq</td>
<td>76</td>
<td>2.2</td>
<td>47.9</td>
<td>0.04</td>
<td>3.10.20.90</td>
<td>d.15.1.1</td>
</tr><tr>
<td>3580</td>
<td>1tit</td>
<td>89</td>
<td>2.1</td>
<td>55.3</td>
<td>0.04</td>
<td>2.60.40.10</td>
<td>b.1.1.4</td>
</tr></table>

Fmaxis obtained within the LJ3 model at the pulling velocity of 0.005AA/τ The first column indicates the ranking of a model protein, the second - the PDB code,and the third - the number of the amino acids that are present in the structure used.Lma denotes the end-to-end distance at which the maximum force arises.λ is the corresponding dimensionless location defined asλ=(Lmax-Ln)/(Lf-Ln),whereLnis the native end-to-end distance and Lf corresponds to full extension. The last two columns give the leading CATH and SCOP codes. The survey is performed based strictly on the PDB-assigned structure codes. It may happen that the structure of a protein has been determined several times and then each of these determinations leads to its own value ofFmax. In this case, one may derive the best estimate either by picking the best resolved structure or by making (weighted) averages over all related structures.

doi:10.1371/journal.pcbi.1000547.t001

the Materials and Methods section. All strong proteins are seen to involve theβ-strands. The peak in the probability distribution for theα-βproteins is observed to be shifted towards the bigger values ofFmaxcompared to the one for the β proteins. At the same time,the high force tail of the distribution for the β proteins is substantially more populated than the corresponding tail for the α proteins.

Figure 2 is similar to Figure 1 in spirit, but now the structures are split into particular ranges of the protein sizes: N between 40and 100 (the dotted line), between 100 and 150 (thin solid line), and between 200 and 250 (the thick solid line). The curve for the range from 150 to 200 is in-between the curves corresponding to neighboring ranges and is not shown in order not to crowd the Figure. The distributions are seen to be shifting to the right when increasing the range of the values of N indicating, that the bigger the number of amino acids, the more likely a protein is to have a large value ofFmax. This observation holds for all classes of the proteins, as evidenced by the insets in Figure 2.

In most cases, the major force peak arises at the begining of stretching where the Go-like model should be applicable most adequately.One can characterize the location of Fmax during the stretching process by a dimensionless parameter λ which is defined in terms of the end-to-end distance, as spelled out in the caption of Table 1. This parameter is equal to 0 in the native state and to 1 in the fully extended state. In 25% of the proteins studied in this


![](https://web-api.textin.com/ocr_image/external/c4dc9087a6650740.jpg)

<!-- PLoS **Computational** Biology | www.ploscompbiol.org 4 October 2009 | **Volume** **5** | **Issue** 10 | e1000547 -->

<!-- Mechanical Strength of 17 134 Model Proteins -->

<!-- α a 17134 0.2 3447 0.2 0.1 β β 0.1 4529 P(Fmax) 0.1 α-β 1tit 0.1 4431 0 0 1 2 3 4 5 7510 1% 0 1 2 3 4 5 6 F {}S [ε/AA] -->
![](https://web-api.textin.com/ocr_image/external/a4374c8e4034dc41.jpg)

**Figure 1. Probability distribution of the maximal forces** **obtained in the set of 17 134 model proteins (solid line).** The shaded histogram corresponds to the 7510 proteins studied in ref. [19]. The insets show similar distributions for the CATH-based classes indicated. Thenumbers underneath the class symbols give the size of the set of the proteins considered.

doi:10.1371/journal.pcbi.1000547.g001

survey,λwas less than 0.25 and in 52% -les than 0.5. There are very few proteins with λ exceeding 0.8.

Table 1 does not include any (non-cysteine-based) knotted proteins. The full list of 17 134 proteins contains 42 such proteins but they come with moderate values ofFmax.However,knotted proteins with N&gt;250may turn out to have different properties.

<!-- α 0.3 0.4 a 0.2 0.2 β β 0.2 b 0.1 P(Fmax) 0.2 α-β C 0.1 0.1 0 2 4 a: 40-100 b: 100-150 c: 200-250 0 0 1 2 3 4 5 6 F max [ε/AA] -->
![](https://web-api.textin.com/ocr_image/external/7bc7ba4b547ff57f.jpg)

**Figure 2. Similar to Figure 1 but for proteins belonging to** **specific ranges of the sequential sizes, as indicated by the** **symbols a,**b, **and** C.

doi:10.1371/journal.pcbi.1000547.g002

# Biological properties of the strongest proteins

A convenient way to learn about the biological properties listed in Tables 1 and S1 is through the Gene Ontology data base [31] which links such properties with the PDB structure codes. The properties are divided into three domains. The first of these is “molecular function” which describes a molecular function of a gene product. The second is "biological processes" and it covers sets of molecular events that have well defined initial and final stages. The third is “cellular component” and it specifies a place where a given gene product is most likely to act.

The results of our findings are summarised in Table 2. It can be seen, that most of the 190 strongest proteins are likely to be found in an extracellular space where conditions are much more reducing than within cells. Larger mechanical stability is advantageous under such conditions. 90 out of the strongest proteins exhibit hydrolase activity. 39 of these 90 are serine-type endopeptidases. These findings seem to be consistent with expectations regarding proteins endowed with high mechanical stability. For instance, proteases, which are well represented in Table 2 should be more stable to prevent self-cleavage.

# CATH-based architectures

The classification of proteins within the CATH (Class, Architecture, Topology, Homology) data base is done semi-automatically by applying numerical algorithms to structures that are resolved better than within 4 Å [24,25]. The four classes of proteins in the CATH system are split into architectures, depending on the overall spatial arrangement of the secondary structures, the numbers of β-sheets in various motifs,and the like. The next finer step in this hierarchical scheme is into topologies and it involves counting contacts between amino acids which are sequentially separated by more than a treshold. The further divisions into homologous superfamilies and then sequence family levels involve studies of the sequential identity.

We have found that only six architectures contribute toFmax larger than 4ϵ/AA. These are ribbons-2.10 (41.8% of the proteins listed in Table1),β-barrels-2.40(8.9%) β-sandwiches-2.60(16.3%),β-rolls-3.10(5.4%)3-layer (aba) sandwiches-3.40 (5.4%), and these with no CATH classification to date (21.8%). The corresponding distributions of forces are shown in the top six panels of Figure 3 and the topologies involved are listed and named in Table 3.

Examples of architectures that are dominant contributors to a low force behavior are the α orthogonal bundle(the right bottom panel of Figure 3), the α up-down bundle,and theβ-roll(the left bottom panel of Figure 3).

## SCOP-based classes and folds

The **SCOP** (Structural Classification of Proteins) data base [27-29] is curated manually and it relies on making comparisons to other structures through a visual inspection. This classification scheme is also hierarchical and the broadest division is into seven classes and three quasi-classes. The classes are labelled a through g and these are as follows: mainly α(a), mainly β (b)),α/β which groups proteins in which helices and β-sheets are interlaced (c),α+βwith the helices and β-sheets grouped into clusters that are separated spatially(),multidomain proteins (e), membrane and cell-surface proteins(f)and small proteins that are dominated by disulphide bridges or the heme metal ligands (g)). The quasi-classes are labelled h throughj and they comprise coiled-coil proteins (h),structures with low resolution (i)), and peptides and short fragments

<!-- PLoS Computational Biology | www.ploscompbiol.org 5 October 2009 | **Volume** **5 |** **Issue** 10 | e1000547 -->

<!--  -->

<!-- Mechanical Strength of 17 134 Model Proteins -->

**Table** 2. Gene Ontology terms for the top 190 proteins.

<table border="1" ><tr>
<td>Domain</td>
<td>GO identifier</td>
<td>Term name</td>
<td>No. of structures</td>
<td>Example</td>
</tr><tr>
<td rowspan="4">Molecular function</td>
<td>GO:0016787</td>
<td>hydrolase activity</td>
<td>90</td>
<td>1f3y</td>
</tr><tr>
<td>GO:0003824</td>
<td>catalytic activity</td>
<td>70</td>
<td>1gvl</td>
</tr><tr>
<td>GO:0004252</td>
<td>serine-type endopeptidase activity</td>
<td>39</td>
<td>1c4p</td>
</tr><tr>
<td>GO:0008083</td>
<td>growth factor activity</td>
<td>25</td>
<td>1bmp</td>
</tr><tr>
<td rowspan="2">Biological process</td>
<td>GO:0006508</td>
<td>proteolytic activity</td>
<td>34</td>
<td>2a7h</td>
</tr><tr>
<td>GO:0007586</td>
<td>digestion</td>
<td>32</td>
<td>1bra</td>
</tr><tr>
<td rowspan="2">Cellular component</td>
<td>GO:0005576</td>
<td>extracellular region</td>
<td>122</td>
<td>1vpf,1aoh</td>
</tr><tr>
<td>GO:0005515</td>
<td>protein binding</td>
<td>70</td>
<td>1bmp</td>
</tr></table>

doi:10.1371/journal.pcbi.1000547.t002

(j). The classes are then partitioned into folds that share spatial arrangement of secondary structures and the nature of their topological interlinking. Folds are then divided into superfamilies (same fold but small sequence identity) and then families (two proteins are said to belong to the same family if their sequence identity is at least 30%). Families are then divided into proteins-a category that groups similar structures that are linked to a similar function. Proteins comprise various protein species.

Each structure assignment comes with an alphanumeric label, as shown in Tables 1, S1, and 4 which reflects the placement in the hierarchy. At the time of our download, there have been 92 972entries in the SCOP data base that are assigned to 34 495 PDB structures. These entries are divided into 3464 families, 1777superfamilies and 1086 unique folds. A given structure may have several entry labels but the dominant assignment is listed first. We use the primary assignment in our studies. The same rule is also applied to the CATH-based codes.

<!-- CATH - architectures 0.2 ribbon 22.10 β-sandwich 2.60 β 343 1832 0.1 0.2 β-barrel 2.40 β α/β-roll 3.10 1540 901 0.1 0 P(Fmax) 0.2 3-layer (aba) α-β complex 3.90 sandwich 3.40 424 1625 0.1 0.3 β-roll2.30 orthogonal 0.2 428 bundle 1.10 2442 0.1 0 0 1 2 3 4 5 1 2 3 4 5 Fmax [e/Å] Fmax [ε/AA] -->
![](https://web-api.textin.com/ocr_image/external/44ef7ee309277b6a.jpg)

**Figure 3. The top six panels show probability distributions of** Fmax**for the architectures that contribute to the pool of** **proteins** **with** **large** **forces.** The architectures are indicated by their names and the accompanying CATH numerical symbol. The numbers underneath the symbols of the architecture inform about the number of cases contributing to the distribution. The bottom two panels show examples of architectures that are predicted to yield only small values ofFmax

doi:10.1371/journal.pcbi.1000547.g003

Figure 4 shows the distributions of forces for the SCOP-based classes of proteins. The results are consistent with the CATH-based classes since the α-β class of CATH basically encompasses the α/βand α+βclasses of SCOP. However, there are proteins which are classified only according to one of the two schemes. Thus there are 4431α-βproteins out of which only the total of 3368 is SCOP-classified as belonging to theα+β and α/β classes. At the same time, the total of the proteins in theα+βandα/βclasses we have is 4795.

It should be noted that the peak in the distribution forα+βis shifted to higher forces by about0.7ϵ/AA from the peak for α/β. At the same time, the zero-force peak is virtually absent inα+β.The SCOP-based classification also reveals that its classg contributes across the full range of forces and, in particular, it may lead to large values of FmaxIt should be noted, as also evidenced by Table 1, that there is a substantial number of strong proteins that has no class assignment.

Figures 5 and 6 refer to the distributions ofFmaxacross specific folds. The first of these presents results for the folds that give rise to the largest forces. The names of such folds are specified in Figure 5. The percentage-wise assessment of the folds contributing to big forces is presented in Table 4. The top contributor is found to be the b.47 fold (SMAD/FHA domain). Figure 6 gives examples of folds that typically yield low forces.

It is interesting to note that distributions corresponding to some folds are distinctively bimodal, as in the case of the SMAD/FHA fold (b.47). This particular fold is dominated by SMAD3 MH2domain (b.47.1.2; 352 structures) which contributes both to the high and low force peaks in the distribution. The remaining domains (b.47.1.1,b47.1.3,and b47.1.4) contribute only to the low force peak. The dynamical bimodality of the b.47.1.2 fold can be ascribed to the fact that the strong subset comes with one extra disulphide bond relative to the weak subset. This extra bond provides substantial additional mechanical stability when stretch-ing is accomplished by the termini. We illustrate sources of this bimodality in the SI (Figure S1) for two proteins from this fold: 1bra which is strong and lelc which is weak. In ref. [18],we have noted that various sets of proteins with identical CATH codes(e.g., 3.10.10) may give rise to bimodal distributions without any dynamical involvement of the disulphide bonds. The reason for this is that even though the contact maps for the two modes are similar, the weaker subset misses certain longer ranged contacts which pin the structure. Mechanical stability is more sensitive to structural and dynamical details than are not provided by standard structural descriptors.

<!-- PLoS Computational Biology | www.ploscompbiol.org 6 October 2009 | Volume 5 | Issue 10 | e1000547 -->

<!--  -->

<!-- Mechanical Strength of 17 134 Model Proteins -->

**Table** 3. CATH classes (C), architectures (A), and topologies (T) contributing to the top strength proteins.

<table border="1" ><tr>
<td>C</td>
<td>A</td>
<td>T</td>
<td>Strong</td>
<td>All</td>
<td></td>
<td>Root name</td>
</tr><tr>
<td>2.</td>
<td></td>
<td></td>
<td>57.3%</td>
<td>26.4%</td>
<td></td>
<td>Mainly β</td>
</tr><tr>
<td></td>
<td>2.10</td>
<td></td>
<td>17.3%</td>
<td></td>
<td>2.0%</td>
<td>Ribbon</td>
</tr><tr>
<td></td>
<td></td>
<td>2.10.70</td>
<td></td>
<td>5.2%</td>
<td>0.1%</td>
<td>Complement Module, domain 1</td>
</tr><tr>
<td></td>
<td>2.40</td>
<td></td>
<td>25.7%</td>
<td></td>
<td>8.9%</td>
<td>β Barrel</td>
</tr><tr>
<td></td>
<td></td>
<td>2.40.10</td>
<td></td>
<td>21.5%</td>
<td>2.9%</td>
<td>Thrombin,subunit H</td>
</tr><tr>
<td></td>
<td>2.60</td>
<td></td>
<td>14.2%</td>
<td></td>
<td>10.6%</td>
<td>Sandwich</td>
</tr><tr>
<td></td>
<td></td>
<td>2.60.40</td>
<td></td>
<td>3%</td>
<td>7%</td>
<td>Immunoglobulin-like</td>
</tr><tr>
<td>3.</td>
<td></td>
<td></td>
<td>26.8%</td>
<td>25.8%</td>
<td></td>
<td>α-β</td>
</tr><tr>
<td></td>
<td>3.10</td>
<td></td>
<td>8.4%</td>
<td></td>
<td>5.2%</td>
<td>Roll</td>
</tr><tr>
<td></td>
<td></td>
<td>3.10.20</td>
<td></td>
<td>2.6%</td>
<td>1.3%</td>
<td>Ubiquitin-like (UB roll)</td>
</tr><tr>
<td></td>
<td></td>
<td>3.10.130</td>
<td></td>
<td>5.7%</td>
<td>1.0%</td>
<td>P-30 Protein</td>
</tr><tr>
<td></td>
<td>3.40</td>
<td></td>
<td>17.9%</td>
<td></td>
<td>9.4%</td>
<td>3-Layer (aba) Sandwich</td>
</tr><tr>
<td></td>
<td></td>
<td>3.40.50</td>
<td></td>
<td>17.9%</td>
<td>5.6%</td>
<td>Rossmann fold</td>
</tr><tr>
<td>X</td>
<td colspan="2"></td>
<td>15.7%</td>
<td>26.6%</td>
<td></td>
<td></td>
</tr></table>

The percentages indicated in the column denode by "Strong" are relative the top 190 proteins listed in Table 1. X corresponds to proteins not listed in CATH. doi:10.1371/journal.pcbi.1000547.t003

# Force clamps

**Shearing** **motif.** The most common type of the force clamp identified in the literature is illustrated in the top left panel of Figure 7 corresponding to the 14th-ranked protein 1c4p.In this case,the strong resistance to pulling is due to a simultaneous shearing of two β-strands which are additionally immobilised by short β-strands that adhere to the two strands. Similar motifs appears in lqqr(15)),1j8s(17), 1j8r(19), 1f3y(20),2pbt(29),2fzl(15), laoh(19),where the number in brackets indicate ranking as shown in Table 1. It is interesting to note that the β-strands responsible for the mechanical clamp in 1j8s and 1j8r display an additional twist. Undoing the twist enhancesFmax. (There is a similar mechanism that seems to be operational in the case of a horseshoe conformation found in ankyrin [32,33]). The force clamps are identified by investigating the effect of removal of various groups of contacts on the value of Fmax [12,18].

There are, however, new types of the force clamps that we observe in the proteins listed in Tables 1 and S1. They arise from entanglements resulting from the presence of the disulphide bonds which cannot be ruptured by forces accessible in the atomic force microscopy. We note that about 2/3 of the proteins listed in Tables 1 and S1 contain the disulphide bonds. Many of these bonds do not carry much of dynamical relevance when pulling by the termini. However, in certain situations they are the essence of the force clamp. The disulphide bonds have been already identified as leading to formation of the cystein knot (CK) motifs

**Table** 4. SCOP classes (C) and folds (F) contributing to the top strength proteins.

<table border="1" ><tr>
<td colspan="6"></td>
</tr><tr>
<td>C F</td>
<td>Strong</td>
<td colspan="2">All</td>
<td>Root name</td>
<td>Description</td>
</tr><tr>
<td>b.</td>
<td>40.5%</td>
<td>22.7%</td>
<td></td>
<td>β</td>
<td></td>
</tr><tr>
<td>b.47</td>
<td></td>
<td>21.5%</td>
<td>2.7%</td>
<td>SMAD/FHA domain</td>
<td>sandwich; 11 strands in 2 sheets; greek-key</td>
</tr><tr>
<td>C.</td>
<td>17.9%</td>
<td>9%</td>
<td></td>
<td>α/β</td>
<td>Mainly parallelβ-sheets (β-α-βunits))</td>
</tr><tr>
<td>c.69</td>
<td></td>
<td>15.7%</td>
<td>0.3%</td>
<td>Pyruvate kinase C-terminal<br>domain-like</td>
<td>3 layers: a/b/a; mixed β-sheet of 5 strands, order 32145, strand 5 is antiparallel to the rest</td>
</tr><tr>
<td>d.</td>
<td>11.05%</td>
<td>18.9%</td>
<td></td>
<td>α+β</td>
<td>Mainly antiparallelβ- sheets (segregated αand β regions)</td>
</tr><tr>
<td>d.5</td>
<td></td>
<td>5.8%</td>
<td>0.9%</td>
<td>RNase A-like</td>
<td>contains long curved β- sheet and 3 helices</td>
</tr><tr>
<td>d.113</td>
<td></td>
<td>2.6%</td>
<td>0.2%</td>
<td>DsrC, the y subunit of dissimilatory sulfite reductase</td>
<td>β(3)-α(5);; meander β-sheet packed against array of<br>helices</td>
</tr><tr>
<td>g.</td>
<td>13.7%</td>
<td>4.9%</td>
<td></td>
<td>Small proteins</td>
<td>Usually dominated by metal ligand, heme, and/ordisulfidebridges</td>
</tr><tr>
<td>g.17</td>
<td></td>
<td>5.2%</td>
<td>0.1%</td>
<td>Necrosis inducing protein 1, NIP1</td>
<td>disulfide-rich fold; all-β; duplication: contains two<br>structural repeats</td>
</tr><tr>
<td>9.18</td>
<td colspan="2">6.3%</td>
<td>0.2%</td>
<td>Trefoil/Plexin domain-like</td>
<td>disulfide-rich fold; common core is α+β with two<br>conserved disulfides</td>
</tr><tr>
<td colspan="3">X 16.3% 27.4%</td>
<td></td>
<td></td>
<td></td>
</tr><tr>
<td colspan="6">X corresponds to proteins not listed in SCOP.</td>
</tr></table>

doi:10.1371/journal.pcbi.1000547.t004

<!-- PLoS Computational Biology | www.ploscompbiol.org 7 October 2009 | Volume 5 | Issue 10 | e1000547 -->

<!--  -->

<!-- Mechanical Strength of 17 134 Model Proteins -->

<!-- SCOP 0.2 d α+β c α/β 3239 1556 0.1 0 b 0.2 β β a α 3898 2538 0.1 P(Fmax) 0 0.6 h coiled-coil 102 f membrane & surface 127 0.4 0.2 0 0.2 g small proteins not in SCOP 848 4707 0.1 0 0 1 2 3 4 5 1 2 3 4 5 Fmax [e/Å] Fmax [e/Å] -->
![](https://web-api.textin.com/ocr_image/external/df1af348165934d0.jpg)

**Figure 4. Distributions of**Fmax**for the SCOP-based classes for** **which there are more than 60 structures that could be used in** **molecular** **dynamics** **studies.** The cases that are not shown are: class e (27 structures), quasi-class i (5 structures), and quasi-class j (52structures). The bottom right panel corresponds to structures which have no assigned SCOP-based structure label. The numbers indicate the corresponding numbers of structures studied. doi:10.1371/journal.pcbi.1000547.g004

[34,35] (such proteins are found in the toxinsof spiders and scorpions) and the cyclic CK motifs [36,37]. Here, we find still another motif-that of the CSK which is similar to that found in slipknotted proteins [38-40] which do not conatin the disulphide

<!-- SCOP b.1 immunoglobulin b.2 common diphteria 0.2 like β-sandwich toxin/transcription 1024 41 0.1 b.3 prealbumin-like b.60 GroES-like 0.2 108 183 0.1 P(Fmax) 0 0.4 b.47 SMAD/FHA domain c.69 Pyruvate kinase 460 C-terminal domain-like 51 0.2 P 0.4 d.15 penicillin-binding g.18 Necrosis inducing protein protein 1,NIP1 227 25 0.2 0 0 1 2 3 4 5 1 2 3 4 5 Fmax [e/Å] Fmax [e/Å] -->
![](https://web-api.textin.com/ocr_image/external/7592afc16d78109c.jpg)

**Figure 5. Distributions of**Fmax**for eight folds that may give rise** **to a large resistance to pulling.** doi:10.1371/journal.pcbi.1000547.g005


![](https://web-api.textin.com/ocr_image/external/faeb24c6e1612130.jpg)

<!-- 0.6 SCOP globin-like long a-hairpin 0.4 392 79 0.2 a.1 a.2 0.4 type I dockerin LEM/SAP HeH motif domain 0.3 211 119 0.2 0.1 a.3 a.4 P(Fmax) 0 lysozyme-like head-to-tail joining 0.2 647 protein W.gpW 80 0.1 d.2 d.9 0.4 mitochondrial dextranase, 0.3 glycoprotein N-terminal domain MAM33-like 218 0.2 290 0.1 d.58 b.34 0 0 1 2 3 4 5 1 2 3 4 5 Fmax [e/Å] Fmax [e/Å] -->
![](https://web-api.textin.com/ocr_image/external/24578a1f1fdedd86.jpg)

**Figure 6. Distribution of**Fmax**for eight folds that are likely to** **yield a small resistance to pulling.** doi:10.1371/journal.pcbi.1000547.g006

bonds. This motif is found in the top 13 proteins. The cysteine loop,knot, and slipknot motifs are shown schematically in the remaining panels of Figure 7. It is convenient to divide these motifs into two categories: shallow (S) and deep (D) (according to the classification used for knotted proteins [41,42]), depending on whether the motif is spanning most of the sequence or is instead localized in its small fraction.

**Shearing connected with a cysteine loop.** In this case,the mechanical clamp arises from shearing between a β- strand belonging to a deep cysteine loop and another strand located

<!-- β β-strands(1c4p) cys loop(1cuz) N C N C cys knot(2g4x) 2g4x) cys loop(1kdm,1qu0) N C C N cys slipknot(1vpf) 57 102 slip-loop 26 knot-loop N 68 61 104 C -->
![](https://web-api.textin.com/ocr_image/external/0ae0dbf69d7534ab.jpg)

**Figure 7. Examples of force clamps found in the top strength** **proteins.** The relevant disulphide bonds are shown in gray shade. The PDB codes of the examples of the proteins that show the particular type of a clamp are indicated. In the case of the CSK, the numbers indicate sequential locations of the amino acids participating in a disulphide bridge in the 13-ranked 1vpf.

doi:10.1371/journal.pcbi.1000547.g007

<!-- PLoS Computational Biology | www.ploscompbiol.org 8 October 2009 | Volume 5 | Issue 10 | e1000547 -->

<!-- **Mechanical Strength of 17 134 Model Proteins** -->

outside the loop (the left bottom panel of Figure 7). Existence of the disulphide bond before the shearing motif allows to decompose direct tension onto the β-strands making the protein resist stretching much more effectively than what would be expected from a simple shearing motif. Additionally, the disulphide bonds prevent an onset of any rotation in the protein conformation which otherwise might form an opportunity for unzipping.This motif appears in 1dzj(40,D) 1vsc(37,D), 1dzk(35,D),1i04 (81,D), 1hqp(83,D), loxm(98,D),2a2g (175,D),2boc(179,D),and many other proteins. The middle panel of Figure 8 gives an example of the corresponding force(F))-displacement (d) pattern as obtained for 1dzj.

**Shearing and dragging out of a cysteine loop.** This motif consists of two parts. The first is formed by a rather small and deep cysteine loop which is located very close to one terminus with the second terminus located across the cysteine loop. The motif arises when almost all of the protein backbone is dragged across the cysteine loop on stretching. A protein structure also contains a few β-strands which get sheared before dragging takes place. This motif is seen in 1kdm(23,1D)1q56(24,D),1qu0(33,D),1f5f(34,D) and this geometry of pulling we call geometry I. It should be pointed out that, in all such cases, pulling by the N terminus takes place within (or very near) the plane formed by the cysteine loop. A small change in such a geometry, e.g. the one arising from pulling not by the last amino acid but by the penultimate bead, may cause getting out of the cystein loop and result in a very different unfolding pathway with a distinctly ddifferent value of Fmax. In this other kind of pulling set up, denoted as geometry II, the loop is bypassed and the resistance to pulling is provided only by the shearing mechanism.

Dragging arises from overcoming steric constraints and generates an additional contribution to the strength of the standard shearing mechanical clamp. By using geometry II and also by eliminating the native contacts between the sheared β-strands we can estimate the topological contribution of the dragging effect on the value ofFmax.For proteins 1kdm,1q56, 1qu0, 1f5f, it comes out to be around 25%. The forceF-d patterns corresponding to these two geometries of pulling are shown in top panel of Figure 9.

<!-- CYSTEINE L LOOP-1 qu0 drag 4 [e/A] shear 2 F 0 200 400 600 CYSTEINE SLIPKNOT- 2h64 4 [e/A] 2 F no contacts 100 200 d [Å] -->
![](https://web-api.textin.com/ocr_image/external/f9b897989665cc41.jpg)

**Figure 8. Examples of the force patterns corresponding to** **proteins with the disulphide bonds.** doi:10.1371/journal.pcbi.1000547.g008

In the survey, there are other proteins which also have disulphide bonds and belong to the 2.60.120.200 category.These proteins have a cysteine which is either very shallow or deep, but is located in the middle of the protein backbone so that there is no possibility to form a longβ-strand. In this case, the dragging effects are much smaller. For instance, for 1pz7(D) and lcpm(S),Fmaxis close to 11ϵ/AA

**Shearing inside of a cysteine knot.** This motif is created by a loosely packed CK (two or more spliced cysteine loops) with at least two parallelβ strands that are present within the knot. Pulling protein by termini exerts tension on the entire CK and thus produces an indirect shearing force on the β-strands inside the entangled part of the protein. In this case, elimination of the native contacts between the β-strands reduces Fmaxonly partially indicating that the mechanical clamp is created also by the CK.A simple CK is also found in 2bzm(42)and many other proteins,e.g. in 2\;g7i(77,\;S)),1hfh103,S),2g4x(136,D)),2g4w(169,D).TheF-d patterns for 2bzm and 2g4x are shown in the bottom panel of Figure 8. More complex structures or higher order CKs (with more than two cystein bonds) can be identified in 1afk(85)),1afl(117), or laqp(135). Inside this group of proteins there are also examples of proteins - 1qoz(88,S) - in which a cysteine loop is braided to a CK by some native contacts.

**Cysteine slipknot force-clamp is observed in the** **strongest** **13** **proteins.** The topstrength protein is 1bmp (bone morphogenic protein) with the predictedFmaxof 10.2ϵ/AA,which should correspond to about 1100pN (see Materials and Methods). This strength should be accessible to standard experiments as the atomic force microscopy has been already

<!-- 6 cys slipknot 1vpf 4 2 6 cys loop [e/A] 4 1bj7 F 2 1dzj 6 cys knot 2bzm(shallow) 4 2 2g4x (deep) 0 100 200 300 d [Å] -->
![](https://web-api.textin.com/ocr_image/external/0fad8d73bf03f8ce.jpg)

**Figure 9. Top: Two trajectories arising in protein 1qu0.** Dragging occurs when the backbone is pulled across the cysteine loop. Shearing occurs when the pull across the cystein loop does not take place. Bottom: The force-displacement pattern corresponding to the CSK force clamp in 2h64 (thick line). The thin line shows the corresponding pattern when one removes the attractive contacts that are slipknot related.

doi:10.1371/journal.pcbi.1000547.g009

PLoS Computational Biology | www.ploscompbiol.org 9 October 2009 | Volume 5 | Issue 10 | e1000547


![](https://web-api.textin.com/ocr_image/external/af26673777564432.jpg)

<!-- Mechanical Strength of 17 134 Model Proteins -->

used to rupture covalent N-C and C-C bonds by forces of 1500and 4500pN respectively [43].

In our discussion, we focus on the 13-ranked 1vpf (a vascular endothelial growth factor) with the predicted 1Fmaxof 5.3ϵ/AA.The CSK motif arises from two loops [40]: the knot-loop and the slip-loop, where the slip-loop can be threaded across the knot-loop. One needs at least three disulphide bonds for this motif to arise.

In the case of the lvpf, the knot-loop is created by the disulphide bonds between amino acids 57 and 102, 61 and 104, and the protein backbone between amino acids 57-61 (GLY,GLY,CYS) and 102-104 (GLU). The slip-loop is created by the protein backbone between sites 61-102 and is stabilized by 12 hydrogen bonds between two parallel β-strands. In the CSK motif, the force peak is due to dragging of a slip-loop through the knot-loop making the native hydrogen contacts only marginally responsible for the mechanical resistance. Thus the force peak arises, to a large extent,from overcoming steric constraints, i.e. it is due to repulsion resulting from the excluded volume. TheF-pattern for this novel type of a force clamp is shown in the top panel of Figure 8. Another example of such a pattern for a CSK is shown in the bottom panel of Figure 9 for the 22nd ranked 2h64 (a human transforming growth factor). The leading role of the steric constraints is verified by checking the reduction of theFmaxwhen all the slipknot-related contacts (inside the slip-loop and between the slip-loop and the knot-loop) are converted to be purely repulsive. As a result of this bond removal, the force peak persists, though it gets shifted and becomes smaller. This is summarized in Table S2 in the SI. It is a new and unexpected result.

Another way to establish the role of the CSK motif is to create the disulphide-deficient mutants, as accomplished experimentally [44]for lvpf. The two mutants, 1mkk (C61A and C104A) and 1mkg (C57A and C102A), have structures similar to lvpf but contain no knot-loops and thus there is no slipknot. Muller et al. [44] show that the mutants' thermodynamic stability is not reduced but their folding capacity is. Our work shows that the mutants have a reduced resistance to pulling compared to 1vpf: Fmax drops from 5.3ϵ/AAto 1.49 and 2.012.01ϵ/AAε for 1mkk and 1mkg respectively.

We note that the CSK topology is a subgroup inside the CK class (represented mostly by 2.10.90.10) and the CSK force clamp need arise for a particular way of pulling. For instance,proteins lafk(68), 1afl(100) or laqp(118) have up to four disulphide bonds and yet the CSK motif does not play any dynamical role in pulling by the terminal amino acids. In the case of the CSK, we observe a formidable dispersion in the values ofFmaxFor example,it ranges between 4.8-5.9,4.1-4.8,and 4.1-5.2ε/A for various trajec-tories in 1vpf, 2h64, and 2c7w respectively.We now examine the CSK geometry in more details.

**Cysteine slipknot motif is distinct from the slipknot motif** **in** **several** **ways.** The left-most panel of Figure 10 shows a slipknot with three intersections at sequential locationsk1k2,and k3.This geometry is topologically trivial since when one pulls by the termini, the apparent entanglement may untie and become a simple line. The entanglement would form the trefoil knot if the k3intersection was removed by redirecting the corresponding segment of the chain (thin line) away from the k1-k3 loop. Such slipknot motifs have been observed in native states of several proteins [38-40]. In contrast, the CSKs are not present in the native state but arise as a result of mechanical manipulation. The middle panel of Figure 10 shows a schematic representation of a native conformation with three cysteine bonds: between i1 and j1i, between i2 and j2, and between i33 and j33. The i-ends of the bonds are counted as being closer to the N-terminus. The three bonds are in a specific arrangement as shown in the panel. In native slipknot

<!-- K3 k3 k2 k1 -->
![](https://web-api.textin.com/ocr_image/external/462a4a0104cd2f13.jpg)

**native cystein knot**

cystein knot under applied force -cystein slipknot

<!-- j2 i2 j3 J i3 i1 -->
![](https://web-api.textin.com/ocr_image/external/07831c87e0721976.jpg)

<!-- i2 j2 J3 j3 RCS Rck i3 i3 J1 j1 i1 -->
![](https://web-api.textin.com/ocr_image/external/4e11b29152d68308.jpg)

**Figure 10. Geometry of a slipknot and a cystein slipknot.** The top panel corresponds to a genuine slipknot. The bottom left panel is a schematic representation of the native geometry that yields the cystein slip-knot on stretching. The resulting cystein slipknot motif is shown in the bottom right panel.

doi:10.1371/journal.pcbi.1000547.g010

particular, the i3-j3 bond must cross the loopi1-i2-j2-j1..This loop consists of two pieces of the backbone(i1-i2.and j2-ji)that are linked to form a closed path by the two remaining cysteine bonds-it is the cysteine knot-loop.The average radius of this loop is denoted byRck

The arrangement shown in the middle panel has no entanglements that could be considered as knots in the topolgical sense. However, on pulling by the termini, the chain segment adjacent to i3 gets threaded through the knot-loop since i3 is rigidly attached to j3, as illustrated in the rightmost panel of Figure 10. Pulling byi3-j3also results in generating another loop - the cysteine slip-loop-since the segment around i3 gets bent strongly to form a cigar like shape with the radius of curvature at the i3-tip denoted byRc This loopextends between i2 and j1. It should be pointed out that the cysteine knot-loop in the CSK is stiff whereas in a slipknotted protein (such as the thymine kinase) its size is variable (as it can be tightened on the protein backbone [40] in analogy to tightening a knot [45] by pulling).

The dynamics of pulling depends of the relationship between RckandRcsas the "cigar" may either go through or get stuck. In the former case a related force peak would arise. If the system was a homogeneous polymer, dragging would be successful whenRck was bigger thanRcs. The corresponding force would be related to the work against the elasticity that was needed to bend the slip-loop to the appropriate curvature. This work is proportional to the square of the curvature. Thus the total elastic energy involved in bending the segmenti2-j1is of order $\oint R^{-2}\;R_{}^{-1}$ [46],where S is the arc distance. Dividing this energy by the distance of pulling would yield an estimate of the force measured if thermal fluctuations were neglected. The geometrical condition for dragging in proteins is more complicated because of the presence of the side groups and the related non-homogeneities and variability across the hydrophobicity scale. The diameter of the “rope” that the knot loop is made of should not exceed the maximum a linear extension,tkof amino acids. Thus the effective inner radius of the knot-loop isRck-tk.Similarly,the size of the outer circle that is tangential to the tightest slip-loop isRcs+ts, wheretsis the thickness of the slip-loop. (Both thicknesses can be considered as being site dependent and including possible hydration layer effects near polar amino acids.) Thus the slip-knot can be driven through the cystein knot-loop provided

<!-- PLoS **Computational Biology** | **www.ploscompbiol.org** 10 October **2009** | **Volume 5 | Issue 10** | e1000547 -->

<!--  -->

<!-- Mechanical Strength of 17 134 Model Proteins -->

$$R_{cs}+t_{s}<R_{ck}-t_{k}\tag{1}$$

In our simulations, the successful threading situations correspond toRckandRcsof around 7 and 3AA. The amino acids in the knot-loop are mostly Gly,Ala,orCyswith their side groups pointing outside of the loop. One may then estimate tk to be about 1.5AA. On the other hand, the linear size of the amino acids in the slip-loop can be determined to be close to 2.5AA. These estimates indicate that Rcs+tscan be very close toRck-tkso the possibility of slipping through the knot-loop is borderline. In fact,slipping might be forbidden within the framework of the tube-picture of proteins [47,48] in which the effective thickness of the tube is considered to be 2.7AA.

The CSK motifs give rise to a force peak in 1vpf, 2h64(22,S), 1rv6(25,S),1waq(26,S),1ru(27S)1tgj(28),2h62(30,S),1tgk(31), 2c7w(38,D),2gyr(39,S), 1lx5(95,D), and many other proteins. In these cases, the typical value ofRck is about 7AA. However, specificity may result in somewhat smaller values ofRck which may cause only smaller segments of the slip-loop to be threaded. If the passage is blocked, there will be no isolated force peak as happens in ltgj and lvpp.

**Types of the force-displacement patterns for proteins** **with** **the** **disulphide** **bonds.** In the case of proteins with very shallow cystein knot, loop or slipknot motifs, F increases very rapidly with d and isolated force peak does not arise (max=0). Such cases are represented,e.g., by 1bmp, 1rnr, 1ld5,and 1wzn where the slipknots are either very tight or the cystein loop is very shallow. In the case of a shallow motif, however, a force peak can sometimes be isolated as in the case of the 13th-ranked protein 1vpf(Figure 8) and in several other proteins, like 1xzg and 1dzk.In this case, the value ofFmaxtakes into account tension on the cystein bonds and it is not obvious whether such a strong elastic background should be subtracted from the value of F when determining Fmaxor not. In this survey, we do not subtract the backgrounds. It should be noted that in our previous surveys we missed the CSK-related force peaks because we attributed the rapid force rises at the end of pullingjust to stretching of the backbone without realizing existence of structure in somesuch rises.

For a deep motif,theF-dpattern may have several small force peaks before the final rise of the force,as observed for2g4s and 1bj7.When the CSK motif is very deep, it usually does not have any influence on the shape of the F-dpattern apart from a much steeper final rising force. Such a situation is seen in the case of, e.g.,1j8r and 1j8s.

# Concluding remarks

This surveys identifies a host of proteins that are likely to be sturdy mechanically. Many of them involve disulphide bridges which bring about entanglements that are complicated topolog-ically such as CSKs and CKs. The distinction between the two is that the former can depart from its native conformation and the latter cannot.

Our survey made use of a coarse grained model so it would be interesting to reinvestigate some of the proteins identified here by all-atom simulations, especially in situations when the CSK is involved. The CSK motifs may reveal different mechanical properties when studied in a more realistic model. Of course, a decisive judgment should be provided by experiment.

The very high mechanical resistance of the CSK proteins should help one to understand their biological function. The superfamily of cysteine-knot cytokines (in class small proteins and fold cystein-knot cytokines)includes families of the transforming growth-factor (TGF)-β and the polypeptide vascular endothelial growth factors (VEGFs) [49,50]. The various members of this superfamily,listed in Table 5, have distinct biological functions. For instance, VEGF-B proteins which regulatethe blood vessel and limphatic angiogenesis bind only to one receptor of tyrosine kinase VEGFR-1. On the other hand, VEGF-A proteins bind to two receptors VEGFR-1 and VEGFR-2. All of these proteins form a dimer structure. The members of this familly are endowed with remarkably similar monomer structures but differ in their mode of dimerisation and thus in their propensity to bind ligands.

**Table** 5. Members of **the** cysteine-knot cytokines superfamilly.

<table border="1" ><tr>
<td>family</td>
<td>domain/complex</td>
<td>PDB</td>
</tr><tr>
<td rowspan="4">VEGF</td>
<td></td>
<td></td>
</tr><tr>
<td>VEGF-A</td>
<td>1vpf*,2vpf*,1cz8,1bj1,1flt,1qty,1fpt, 1mjv,1mkg,1mkk</td>
</tr><tr>
<td>VEGF-B</td>
<td>2c7w</td>
</tr><tr>
<td>VEGF-F</td>
<td>1wq9,1wq8,1rv6,1fzv</td>
</tr><tr>
<td>TGF</td>
<td></td>
<td></td>
</tr><tr>
<td rowspan="3"></td>
<td>BMP7/ActRII</td>
<td>1lx5,1lxi,1m4u,1bmp</td>
</tr><tr>
<td>BMP2/IA</td>
<td>1reu,1rew, 2es7,3bmp*</td>
</tr><tr>
<td>BMP2 ternary ligand-receptor complex</td>
<td>2h62,2h64</td>
</tr><tr>
<td rowspan="2"></td>
<td>human arthemine/GFRbeta3</td>
<td>1tgj,1tgk</td>
</tr><tr>
<td>human arthemine/GFRalpha3</td>
<td>2gh0,2gyz</td>
</tr><tr>
<td>human and differential factor 5</td>
<td></td>
<td>1waq,2bhk</td>
</tr></table>

VEGF stands for vascular endothelial growth factor, BMP for bone morphogenetic protein, and TGF for transforming growth factor. The star * indicates uncomplexed proteins.

doi:10.1371/journal.pcbi.1000547.t005

<!-- PLoS Computational Biology | www.ploscompbiol.org 11 October 2009 | Volume 5 | Issue 10 | e1000547 -->

<!--  -->

<!-- **Mechanical** **Strength** of 17 134 Model Proteins -->

Additionally, all dimers posses almost the same a cyclic arrangement of cysteine residues which are involved in both intra- and inter-chain disulphide bonds. These inter-chain disulphide bonds create the knot and slip-loops, where the intra-chain disulphide bonds give rise to a CSK motif when the slip-loop is gets dragged acrros the knot-loop upon pulling.

It has been shown experimentally [51] that such cysteine related connectivities bring the key residues involved in receptor recogni-tion into close proximity of each other.They also provide a primary source of stability of the monomers due to the lack of other hydrogen bonds between two beta strands at the dimer interface.

The non trvial topologial connection between the monomers allow for mechanical separation of two monomers by a distance of about half of the size of the slip-loop. Our results suggest,however, that the force needed for the separation may be too high to arise in the cell.

# Materials and Methods

The input to the ddynamical modeling is provided by a PDB-based structures. The structure files may often contain several chains. In this case, we consider only the first chain that is present in the PDB file. Likewise, the first NMR determined structure is considered. If a protein consists of several domains, we consider only the first of them.

The modeling cannot be accomplished if a structure has regions or strings of residues which are not sufficiently resolved experimen-tally.Essentially aIl structure-disjoint proteins have been excluded for our studies. Exceptions were made for the experimentally studied scaffoldin laoh and for proteins in which small defects in the established structure (such as missing side groups) were confined within cystein loops and were thus irrelevant dynamically.In these situations, the missing contacts have been added by a distance based criterion [23] in which the treshold was set at7.5AA.Among the test used to weed out inadequate structures involved determining distances between the consecutive Cα atoms. A structure was rejected if these distances were found to be outside of the range of 3.6-3.95AA..The exception was made for prolines, which in its native state can accommodate the cis conformation. In that case,the distance between a prolineCxand its subsequent amino acid usually falls in the range between 2.8 and 3.85AA. For a small group of proteins which slipped through our structure quality checking procedure,but were found to be easily fixed (e.g. 1f5f, 1fy8,and 2f3c) , we used publicly avialable software BBQ [52] to rebuild locations of the missing residues. A limited accuracy of this prediction procedure seems to be adequate for our model due to its the coarse-grained nature.

The modeling of dynamics follows our previous implementa-tions [11,12,18] within modelLJ2except that the contact map is as in ref.[19],i.e.with thei,i+contacts excluded. There is also a difference in description of the disulphide bonds. In refs. [14,19] they were treated as an order-of-magnitude enhancement of the Lennard-Jones contacts in all proteins. In ref. [18] the different treatment of the disulphide bonds was applied to the proteins that were found to be strong mechanically without any enhancements. Here, on the other hand, we consider such bonds as harmonic in all proteins, in analogy to the backbone links between the consecuitive Cαs. The native contactsare,described by the Lennard-Jones potential $V^{6-12}=4\varepsilon \left[\left(\frac {σ_{ij}}{r_{ij}}\right)^{12}-\left(\frac {σ_{ij}}{r_{ij}}\right)^{6}\right]$  where rijis the distance between theCx′sin amino acids iandj whereas σijis determined pair-by-pair so that the minimum in the potential is located at the experimentally established native distance.The non-native contacts are repulsive belowrijof 4 -7

The implicit solvent is described by the Langevin noise and damping terms. The amplitude of the noise is controlled by the temperature, T. All simulations were done atkBT=03ϵwhere kB is the Boltzmann constant. Newton's equations of motion are solved by the fifth order predictor-corrector algorithm. The model is considered in the overdamped limit so that the characteristic time scale, τ, is of order 1 ns as argued in refs. [6,53]. Stretching is implemented by attaching an elastic spring to two amino acids. The spring constant used has a value of0.12ϵ/AA2which is close to the elasticity of experimental cantilevers. One of the springs is anchored and the other spring is moving with a constant speed,vp.Choices in the value of the spring constant have been found to affect the look of the force-displacements patterns and thus the location of the transition state [54,55],but not the values ofFmax [10,12,18].

The dependence on vp is protein-dependent and it is approximately logarithmic invpas evidenced by Figure 11 for several strong proteins. The logarithmic dependence has been demonstrated experimentally, for instance, for polyubiquitin [56,57]. Fmax=pln(v/v0)+q. The approximate validity of this relationship is demonstrated in Figure 11 for three proteins with big values ofFmax.We observe that the larger the value ofFmax,the bigger probability that the dependence on vp is large. When we make a fit to Fmax=pln(v/v0)+q for lvpf, 1c4p, and 1j8s, we get the parameter p to be equal to 0.39±0.110.17±0.03,and C0.04±0.02ϵ/AA respectively (the values of q are 7.42±0.63,5.85±0.16,and4.96±0.08ϵ/AA correspondingly). However,some strong proteins may have p to be as low as 0.04.

When making the survey,we have usedvpof 0.005AA/τand stretching was accomplished by attaching the springs to the terminal amino acids (there is an astronomical number of other choices of the attachment points).

In order to estimate an effective experimental value of the energy parameter 3, we have correlated the theoretical values of Fmax with those obtained experimentally. The experimental data points used in ref. [14] have been augmented by entries pertaining to lemb (117-182), lemb (182-212) [58] (where the numbers in brackets indicate the amino acids that are pulled) and laoh,lglk,

<!-- xx x 5.5 1vpf ☐ 1c4p x [e/A] x x ☐ 5 x ☐ ☐ 0 F max ø □□□ x x F x # x ☐ 4.5 1j8s -8 -6 -4 ln(vp/v0) -->
![](https://web-api.textin.com/ocr_image/external/eb693f5f9182d4e1.jpg)

**Figure 11. Dependence of**Fmax**and the pulling velocity for the** **proteins** **indicated.** vo corresponds to 1AA/τ which is of order 108 nm/s. The data for several top strength proteins are shown. doi:10.1371/journal.pcbi.1000547.g011

<!-- PLoS Computational Biology | www.ploscompbiol.org 12 October 2009 | Volume 5 | Issue 10 | e1000547 -->

<!--  -->

<!-- Mechanical Strength of 17 134 Model Proteins -->

and lamu [23].The full list of the experimental entries is provided by Table 6. Unlike the previous plots [14] thatcross correlate the experimental and theoretical values ofFmax,we now extrapolate the theoretical forces to the values that should be measured at the pulling speeds that are used experimentally. We assume that the unit of speed, v0=1AA/τ,is of order 1.1AA/ns and consider 10speeds to make a fitto the logarithmic relationship. The values of parameters p and q for the proteins studied experimentally are listed in Table 6.

The main panel of Figure 12 demonstrates the relationship between the extrapolated theoretical and experimental values of Fmax.The best slope, indicated by the solid line, corresponds to the slope of 0.0091. The inverse of this slope yields 110 pN as an effective equivalent of the theoretical force unit of ϵ/AA. The

**Table** 6. The experimental and **theoretical** **data** **on** **stretching** of proteins.

<table border="1" ><tr>
<td>n</td>
<td>PDB</td>
<td>Fmaxe[pN]</td>
<td>vp[\;nm/s]</td>
<td>Fmaxt[ϵ/AA]</td>
<td>Fmaxte[ϵ/AA]</td>
<td>p[ϵ/AA]</td>
<td>q[ϵ/AA]</td>
<td>Note</td>
<td>Ref.</td>
</tr><tr>
<td>1</td>
<td>1tit</td>
<td>204+/-30</td>
<td>600</td>
<td>2.15</td>
<td>1.85</td>
<td>0.040</td>
<td>2.335</td>
<td>127*8</td>
<td>[21,22]</td>
</tr><tr>
<td>2</td>
<td>1nct</td>
<td>210+/-10</td>
<td>500</td>
<td>2.4+/-0.2</td>
<td>1.48</td>
<td>0.100</td>
<td>2.703</td>
<td>154-159</td>
<td>[59,60]</td>
</tr><tr>
<td>3</td>
<td>1g1c</td>
<td>127+/-10</td>
<td>600</td>
<td>2.3+/-0.2</td>
<td>2.23</td>
<td>0.038</td>
<td>2.680</td>
<td>15 titin</td>
<td>[61]</td>
</tr><tr>
<td>4</td>
<td>1b6i</td>
<td>64+/-30</td>
<td>1000</td>
<td>1.2</td>
<td>0.74</td>
<td>0.084</td>
<td>1.710</td>
<td>T4 lysozyme(21-141)</td>
<td>[62]</td>
</tr><tr>
<td>5</td>
<td>1aj3</td>
<td>68+/-20</td>
<td>3000</td>
<td>1.23</td>
<td>0.71</td>
<td>0.107</td>
<td>1.830</td>
<td>spectrin R16</td>
<td>[63]</td>
</tr><tr>
<td>6</td>
<td>1dqv</td>
<td>60+/-15</td>
<td>600</td>
<td>1.5</td>
<td>0.58</td>
<td>0.147</td>
<td>2.349</td>
<td>calcium binding C2A</td>
<td>[64]</td>
</tr><tr>
<td>7</td>
<td>1rsy</td>
<td>60+/-15</td>
<td>600</td>
<td>1.7+/-0.2</td>
<td>1.48</td>
<td>0.040</td>
<td>1.962</td>
<td>calcium binding C2A</td>
<td>[64]</td>
</tr><tr>
<td>8</td>
<td>1byn</td>
<td>60+/-15</td>
<td>600</td>
<td>1.4</td>
<td>1.18</td>
<td>0.066</td>
<td>1.981</td>
<td>calcium binding C2A</td>
<td>[64]</td>
</tr><tr>
<td>9</td>
<td>1cfc</td>
<td>&lt;20</td>
<td>600</td>
<td>0.55</td>
<td>0.37</td>
<td>0.052</td>
<td>0.997</td>
<td>calmodulin</td>
<td>[64]</td>
</tr><tr>
<td>10</td>
<td>1bni</td>
<td>70+/-15</td>
<td>300</td>
<td>1.4,1.7</td>
<td>1.06</td>
<td>0.044</td>
<td>1.606</td>
<td>barnase/i27</td>
<td>[65]</td>
</tr><tr>
<td>11</td>
<td>1bnr</td>
<td>70+/-15</td>
<td>300</td>
<td>1.05</td>
<td>0.71</td>
<td>0.053</td>
<td>0.053</td>
<td>barnase/i27</td>
<td>[65]</td>
</tr><tr>
<td>12</td>
<td>1bny</td>
<td>70+/-15</td>
<td>300</td>
<td>1.1,1.3</td>
<td>0.65</td>
<td>0.046</td>
<td>0.046</td>
<td>barnase/i27</td>
<td>[65]</td>
</tr><tr>
<td>13</td>
<td>1hz6</td>
<td>152+/-10</td>
<td>700</td>
<td>3.5</td>
<td>2.79</td>
<td>0.064</td>
<td>3.542</td>
<td>protein L</td>
<td>[66]</td>
</tr><tr>
<td>14</td>
<td>1hz5</td>
<td>152+/-10</td>
<td>700</td>
<td>2.8</td>
<td>2.22</td>
<td>0.104</td>
<td>0.104</td>
<td>protein L</td>
<td>[66]</td>
</tr><tr>
<td>15</td>
<td>2ptl</td>
<td>152+/-10</td>
<td>700</td>
<td>2.2+/-0.2</td>
<td>1.88</td>
<td>0.045</td>
<td>0.045</td>
<td>protein L</td>
<td>[66]</td>
</tr><tr>
<td>16</td>
<td>1ubq</td>
<td>230+/-34</td>
<td>1000</td>
<td>2.32</td>
<td>1.47</td>
<td>0.134</td>
<td>3.019</td>
<td>ubiquitin</td>
<td>[57]</td>
</tr><tr>
<td>17</td>
<td>1ubq</td>
<td>85+/-20</td>
<td>300</td>
<td>0.9</td>
<td>0.72</td>
<td>0.083</td>
<td>1.779</td>
<td>ubiquitin(K48-C)*(2-7)</td>
<td>[56,57]</td>
</tr><tr>
<td>18</td>
<td>1emb</td>
<td>350+/-30</td>
<td>3600</td>
<td>5.15+/-0.4</td>
<td>4.16</td>
<td>0.121</td>
<td>5.403</td>
<td>GFP(3-132)</td>
<td>[67]</td>
</tr><tr>
<td>19</td>
<td>1emb</td>
<td>407+/-45</td>
<td>12000</td>
<td>5.15+/-0.4</td>
<td>4.30</td>
<td>0.121</td>
<td>5.403</td>
<td>GFP(3-132)</td>
<td>[68]</td>
</tr><tr>
<td>20</td>
<td>1emb</td>
<td>346+/-46</td>
<td>2000</td>
<td>5.15+/-0.4</td>
<td>4.09</td>
<td>0.121</td>
<td>5.403</td>
<td>GFP(3-132)</td>
<td>[68]</td>
</tr><tr>
<td>21</td>
<td>1emb</td>
<td>117+/-19</td>
<td>3600</td>
<td>2.3,4.3</td>
<td>1.91</td>
<td>0.050</td>
<td>2.427</td>
<td>GFP(3-212)</td>
<td>[68]</td>
</tr><tr>
<td>22</td>
<td>1emb</td>
<td>127+/-23</td>
<td>3600</td>
<td>2.2+/-0.2</td>
<td>1.51</td>
<td>0.164</td>
<td>3.197</td>
<td>GFP(132-212)</td>
<td>[68]</td>
</tr><tr>
<td>23</td>
<td>1emb</td>
<td>548+/-57</td>
<td>3600</td>
<td>3.5+/-0.1</td>
<td>2.89</td>
<td>0.142</td>
<td>4.347</td>
<td>GFP(117-182)</td>
<td>[58]</td>
</tr><tr>
<td>24</td>
<td>1emb</td>
<td>356+/-61</td>
<td>3600</td>
<td>3.2+/-0.2</td>
<td>2.94</td>
<td>0.075</td>
<td>3.709</td>
<td>GFP(182-212)</td>
<td>[58]</td>
</tr><tr>
<td>25</td>
<td>1emb</td>
<td>104+/-40</td>
<td>3600</td>
<td>2.3+/-0.2</td>
<td>1.26</td>
<td>0.236</td>
<td>3.683</td>
<td>GFP(N-C)</td>
<td>[67]</td>
</tr><tr>
<td>26</td>
<td>1fnf</td>
<td>75+/-20</td>
<td>3000</td>
<td>1.6,1.8</td>
<td>1.70</td>
<td>0.130</td>
<td>3.069</td>
<td>Fniii-10</td>
<td>[69,70]</td>
</tr><tr>
<td>27</td>
<td>1ttf</td>
<td>75+/-20</td>
<td>600</td>
<td>0.7,1.2</td>
<td>0.99</td>
<td>0.006</td>
<td>1.071</td>
<td>Fniii-10</td>
<td>[71]</td>
</tr><tr>
<td>28</td>
<td>1ttg</td>
<td>75+/-20</td>
<td>600</td>
<td>0.7,1.0</td>
<td>0.17</td>
<td>0.099</td>
<td>1.365</td>
<td>Fniii-10</td>
<td>[71]</td>
</tr><tr>
<td>29</td>
<td>1fnh</td>
<td>124+/-18</td>
<td>600</td>
<td>1.8</td>
<td>1.10</td>
<td>0.127</td>
<td>2.635</td>
<td>Fniii-12</td>
<td>[70]</td>
</tr><tr>
<td>30</td>
<td>1fnh</td>
<td>89+/-18</td>
<td>600</td>
<td>1.4,1.7</td>
<td>1.10</td>
<td>0.127</td>
<td>2.635</td>
<td>Fniii-13</td>
<td>[70]</td>
</tr><tr>
<td>31</td>
<td>1oww</td>
<td>220+/-31</td>
<td>600</td>
<td>2.1+/-0.2</td>
<td>2.01</td>
<td>0.024</td>
<td>2.300</td>
<td>FNiii-1</td>
<td>[70]</td>
</tr><tr>
<td>32</td>
<td>1ten</td>
<td>135+/-40</td>
<td>500</td>
<td>1.7</td>
<td>1.53</td>
<td>0.026</td>
<td>1.857</td>
<td>TNFNiii-3</td>
<td>[70,72]</td>
</tr><tr>
<td>33</td>
<td>1pga</td>
<td>190+/-20</td>
<td>400</td>
<td>2.4,+/-0.2</td>
<td>2.50</td>
<td>0.001</td>
<td>2.761</td>
<td>protein G</td>
<td>[73]</td>
</tr><tr>
<td>34</td>
<td>1gb1</td>
<td>190+/-20</td>
<td>400</td>
<td>1.65+/-0.2</td>
<td>1.69</td>
<td>0.045</td>
<td>2.237</td>
<td>protein G</td>
<td>[73]</td>
</tr><tr>
<td>35</td>
<td>1aoh</td>
<td>480+/-14</td>
<td>400</td>
<td>4.3+/-0.2</td>
<td>3.69</td>
<td>0.119</td>
<td>0.119</td>
<td>scaffoldin c7A</td>
<td>[23]</td>
</tr><tr>
<td>36</td>
<td>1g1k</td>
<td>425+/-9</td>
<td>400</td>
<td>3.9+/-0.01</td>
<td>3.22</td>
<td>0.028</td>
<td>4.106</td>
<td>scaffoldin c1C</td>
<td>[23]</td>
</tr><tr>
<td>37</td>
<td>1anu</td>
<td>214+/-8</td>
<td>400</td>
<td>3.3+/-0.03</td>
<td>2.55</td>
<td>0.060</td>
<td>3.224</td>
<td>scaffoldin c2A</td>
<td>[23]</td>
</tr><tr>
<td>38</td>
<td>1qjo</td>
<td>15+/-10</td>
<td>600</td>
<td>1.2</td>
<td>1.25</td>
<td>0.029</td>
<td>1.601</td>
<td>eE2lip3(N-C)</td>
<td>[26]</td>
</tr></table>

Fmaxedenotes the experimentally measured value ofFmaxas reported in the reference stated in the last column.vpdenotes the experimental pulling speed used.Fmaxti the value of the maximal force obtained in our simulation within the LJ3 model. They were performed atvp=0005AA/τFmaxcorresponds to the theoretical estimate ofFmaxwhen extrapolated to the experimental speeds. The extrapolation assumes the approximate logarithmic dependenceFmax=pln(v/vo)+q, where v0is 1AA/τ. 10 speeds were used to determine the values ofdand b in analogy to the procedure illustrated in Figure 11 The values ofdand q are provided in columns 7 and 8 of the Table respectively. The first column indicates the corresponding symbol that is used in Figure 12. doi:10.1371/journal.pcbi.1000547.t006

<!-- PLoS Computational Biology | www.ploscompbiol.org 13 October 2009 | Volume 5 | Issue 10 | e1000547 -->

<!--  -->

<!-- Mechanical Strength of 17 134 Model Proteins -->

<!-- LJ3 4 2b8 19 extrapolated 35 36 3 24 23 [e/A] 13 38 37 314 200 400 Xeu 2 21 31 15 FPe 26 34 4 LJ3 7 $\frac {32}{22}$ 216 38 25 1 103029 27 2 517 01 2 9 28 0 100 200 300 400 500 $\hat {\mathfrak {T}}^{\circ }$ max [pN] -->
![](https://web-api.textin.com/ocr_image/external/0f9f9d6b10f8aade.jpg)

**Figure 12. Theoretical**Fmatex**extrapolated to the pulling speeds** **used experimentally vs. the corresponding experimental** **value,**Fmax. The solid line indicates the best slope of 1/(110pN).The dotted line corresponds to the previous result of 1/(71 pN) obtained in ref. [14] where no exptrapolation was made. The inset shows a similar plot in which the extrapolation is not implemented (denoted asFmaxin Table 6). The list of the proteins used is provided by Table 6. It comprises almost all cases considered in ref. [14] but it also includes the recent data points obtained for the scaffoldin proteins [23] and the GFP [58]. The numerical symbols used in the Figure match the listing number in Table 6.

doi:10.1371/journal.pcbi.1000547.g012

Pearson correlation coefficient,R2is 0.832,the rms percent error, reis 1.02, and the Theil U coefficient (discussed in ref. [14]) is 0.281. The inset show a similar plot obtained when the extrapolation to the experimental speeds is not done. The resulting unit of the force would be equivalent to 110 pN which differs form the previous estimate of 71p(shown by the dotted line in the main panel) because of the inclusion of the newly measured proteins and implementation of the extrapolation procedure. The statistical measures of error here are R2=0851,re=0.37,and U=0..251. These measures are better compared to the case with the extrapolation because the

## References

1. Carrion-Vazquez M, Cieplak M, Oberhauser AF (2009) Protein mechanics at the single-molecule level. In Encyclopedia of Complexity and Systems Science: 7026-7050. Editor-in-chief R. A. Meyers, Springer, New York, ISBN: 978-0-387-75888-6.

2. Lu H, Schulten K (1999) Steered molecular dynamics simulation of conformational changes of immunoglobulin domain I27 interpret atomic force microscopy observations. J Chem Phys 247: 141-153.

3. Paci E, Karplus M (1999) Forced Unfolding of Fibronectin Type 3 Modules: An Analysis by Biased Molecular Dynamics Simulations. J Mol Biol 288: 441-459.

4. Berman H M, Westbrook J, Feng Z, Gilliland G, Bhat TN, et al. (2000) The Protein Data Bank. Nucl Acids Res 28: 235-242.

5. Abe H, Go N (1981) Noninteracting local-structure model of folding and unfolding transition in globular proteins. II. Application to two-dimensional lattice proteins. Biopolymers 20: 1013-1031.

6. Veitshans T, Klimov D, Thirumalai D (1997) Protein folding kinetics: time scales, pathways, and energy landscapes in terms of sequence-dependent properties. Folding Des 2: 1-22.

7. Hoang TX, Cieplak M (2000) Sequencing of folding events in Go-like proteins. J Chem Phys 113:8319-8328.

extrapolation procedure itself brings in additional uncertainties. Nevertheless, implementing the procedure seems sounder physi-cally.The spread between these various effective units of the force suggests an error bar of order 30 pN on the currently best value of 110pN.

## Supporting Information

**Figure** **S1** (a) Structure of trypsin 1bra(N=245).The mechan-ically crucial disulphide bond between sites 128 and 232 is highlighted in red. (b) Structure of elastase lelc(N=255)which belongs to the same fold b.47.1.2 as 1bra. This structure does not contain two disulphide bonds that 1bra does. (c) The force-displacement plot for 1bra. Fmax corresponds to 3.7ϵ/AA. The thinner line is obtained when the 128-232 disulphide bond is eliminated -Fmax drops to 2.7ϵ/AA. When one more disulphide bond is cut, stretching continues to distances shown in panel (d) without affectingFmax(d) The force-displacement plot for lelc. The correspondingFmax is 2.0ϵ/AA In the case of lelc,stretching results in the terminal helix pulling β strands from the inside of the protein and thus causing the inner β-barrel to unfold. If the case of 1bra (with the disulphide bridge), the terminal helix pulls the neighbouring loop. After this event, resistance grows linearly and forms one major force peak. After the peak, the whole structure opens suddenly, rupturing contacts between strands in the β-barrel and in the neighbouring loops.

Found at: doi:10.1371/journal.pcbi.1000547.s001 (4.07 MB EPS)

**Table** **S1** Continuation of Table 1 of the main text.

Found at: doi:10.1371/journal.pcbi.1000547.s002 (0.04 MB PDF)

**Table** **S2** Identification of a mechanical clampFmaxfor selected proteins.

Found at: doi:10.1371/journal.pcbi.1000547.s003 (0.02 MB PDF)

## Acknowledgments

The idea of making surveys of proteins using Go-like models arose in a very stimulating discussion with J. M. Fernandez in 2005. More recent discussions and suggestions by M. Carrion-Vazquez, particularly about the cysteine knots, are warmly appreciated.

## Author Contributions

Conceived and designed the experiments: MS JIS MC. Performed the experiments: MS JIS MC. Analyzed the data: MS JIS MC. Contributed reagents/materials/analysis tools: MS JIS MC. Wrote the paper: MS JIS MC.

8. Clementi C, Nymeyer H, Onuchic JN (2000) Topological and energetic factors: what determines the structural details of the transition state ensemble and "on-route" intermediates for protein folding? An investigation for small globular proteins. J Mol Biol 298: 937-953.

9. Karanicolas J, Brooks CL III (2002) The origins of asymmetry in the folding transition states of protein L and protein G. Protein Science 11: 2351-2361.

10. Cieplak M, Hoang TX, Robbins MO (2002) Folding and stretching in a Go-like model of titin. Proteins: Struct Funct Bio 49: 114-124.

11. Cieplak M, Hoang TX (2003) Universality classes in folding times of proteins. Biophys J 84:475-488.

12. Cieplak M, Hoang TX, Robbins MO (2004) Thermal effects in stretching of Go-like models of titin and secondary structures. Proteins: Struct Funct Bio 56: 285-297.

13. Tozzini V, Trylska J,Chang C, McCammon JA (2007) Flap opening dynamics in HIV-1 protease explored with a coarse-grained model. J Struct Biol 157: 606-615.

14. Sulkowska JI, Cieplak M (2008) Selection of optimal variants of Go-like models of proteins through studies of stretching. Biophys J 95:3174-3191.

15. TsaiJ,Taylor R, Chothia C, Gerstein M (1999) The packing density in proteins: Standard radii and volumes. J Mol Biol 290: 253-266.

<!-- PLoS **Computational** Biology | www.ploscompbiol.org 14 **October 2009 | Volume 5 | Issue 10 | e1000547** -->

<!--  -->

<!-- **Mechanical Strength** of 17 134 Model Proteins -->

16. Settanni G,Hoang TX, Micheletti C, Maritan A (2002) Folding pathways of prion and doppel. Biophys J 83: 3533-3541.

17. Sobolev V, Sorokin A, Prilusky J, Abola EE, Edelman M (1999) Automated analysis of interatomic contacts in porteins. Bioinformatics 15: 327-332.

18. Sulkowska JI, Cieplak M (2007) Mechanical stretching of proteins-a theoretical survey of theProtein Data Bank. J Phys Cond Mat 19: 283201.

19. Sulkowska JI,Cieplak M (2008) Stretching to understand proteins-A survey o the Protein Data Bank. Biophys J 94: 6-13.

20. Cieplak M, Sulkowska JI (2009) Tests of the Structure-based models of proteins. Acta Phys Polonica 115: 441-445.

21. Rief M, Gautel M, Oesterhelt F, Fernandez JM, Gaub HE (1997) Reversible unfolding of individual titin immunoglobulin domains by AFM. Science 276: 1109-1112.

22. Carrion-Vasquez M, Oberhauser AF, Fowler SB, Marszalek PE, Broedel PE, et al. (1999) Mechanical and chemical unfolding of a single protein: a comparison. Proc Natl Acad Sci U S A 96: 3694-3699.

23. Valbuena A, Oroz J, Hervas R, Vera AM, Rodriguez D, et al. (2009) On the remarkable robustness of scaffoldins. Proc Natl Acad Sci U S A - in press (to arrive on-line on July 27).

24. Orengo CA, Michie AD,Jones S,Jones DT, Swindells MB, et al. (1997) CATH-A hierarchical classification of protein domain structures. Structure 5: 1093-108.

25. Pearl F, Todd A, Sillitoe I, Dibley M, Redfern O, et al. (2005) The CATH Domain Structure Database and related resources Gene3D and DHS provide comprehensive domain family information for genome analysis. Nucl Acid Res 33:D247-51.

26. Brockwell DJ, Paci E, Zinober RC, Beddard G, Olmsted PD, Smith DA, Perham RN, Radford SE (2003) Pulling geometry defines mechanical resistance of β-sheet protein. Nat Struct Biol 10: 731-737.

27. Murzin AG, Brenner SE, Hubbard T, Chothia C (1995) SCOP: a structural classification of proteins database for the investigation of sequences and structures.J Mol Biol 247: 536-40.

28. Andreeva A, Howorth DD, Chandonia J-M, Brenner SE, Hubbard TJP, et al. (2008) Data growth and its impact on the SCOP database: new developments. Nucl Acid Res 36: D419-25.

29. Lo Conte L, Brenner SE, Hubbard TJP, Chothia C, Murzin AG (2002) SCOP database in 2002:refinements accommodate structural genomics. Nucl Acid Res 30:264-7.

30. Sulkowska JI, Cieplak M (2008) Stretching to understand proteins - a survey of the Protein Data Bank (Retraction). Biophys J 95: 5487-5487.

31. The Gene Ontology Consortium (2000) gene ontology: tool for the unification of biology.Nat Genet 25:25-29.

32. Lee G, Abdi K,Jiang Y, Michaely P, Bennett V, et al. (2006) Nanospring behavior of ankyrin repeats. Nature 440: 246-249.

33. Li LW,Wetzel S, Pluckthun A, Fernandez JM (2006) Stepwise unfolding of ankyrin repeats in a single protein revealed by atomic force microscopy. Biophys J 90:30-32.

34. Craik DJ, Dally NL, Waine C (2001) The cysteine knot motif in toxins and implications for drug design. Toxicon 39: 43-60.

35. Gruber CW,Cemazar M,Anderson MA, Craik DJ (2007) Insecticidal plant cyclotides and related cysteine knot toxins. Toxicon 49: 561-575.

36. Craik DJ, Daly NL, Bond TJ, Waine C (1999) Plant cyclotides: A unique family of cyclic and knotted proteins that defines the cyclic cystine knot structural motif. J Mol Biol 294:1327-1336.

37. Rosengren KJ, Daly NL, Plan MR, Waine C, Craik DJ (2003) Twists, knots,and rings in proteins - Structural definition of the cyclotide framework. J Biol Chem 278:8606-8616.

38. Yeates TO, Norcross TS, King NP (2007) Knotted and topologically complex proteins as models for studying folding and stability. Curr Opinion in Chem Biol 11:595-603.

39. Taylor WR (2007) Protein knots and fold complexity: Some new twists. Comp Biol and Chem 31: 151.

40. Sulkowska JI, Sulkowski P, Onuchic JN, Jamming proteins with slipknots and their free energy landscapes. submitted (2009).

41. Virnau P,Mirny LA, Kardar M (2006) Intricate Knots in Proteins: Function and Evolution. PLOS Comp Biol 2:1074-1079.

42. Sulkowska JI, Sulkowski P, Szymczak P, Cieplak M (2008) Stabilizing effect of knots on proteins. Proc Natl Acad Sci U S A 105: 19714-19719.

43. Grandbois M, Beyer M, Rief M, Clausen-Schaumann H,Gaub H (1999)How Strong Is a Covalent Bond? Science 283: 1727-1730.

44. Muller YA,Heiring C, Misselwitz R, Welfle K, WeIfle H (2002)The cystine knot promotes folding and not thermodynamic stability in vascular endothelial growth factor.J Biol Chem 277:43410.

45. Sulkowska JI,Sulkowski P, Szymczak P, Cieplak M (2008) Tightening of knots in the proteins. Phys Rev Lett 100: 058106.

46. Landau LD,Lifshitz EM (1986)Theory of Elasticity-Course of Theoretical Physics, 3rd edition. Oxford: Butterworth-Heinemann.

47. Banavar JR, Hoang TX, Maritan A, Seno F, Trovato A (2004) Unified perspective on proteins: A physics approach. Phys Rev E 70: 041905.

48. Lezon TR, Banavar JR, Maritan A (2006) The origami of life. J Phys: Cond Matter 18:847-888.

49.Iyer S,Scotney PD, Nash AD, Acharya KR (2006) Crystal Structure of Human Vascular Endothelial Growth Factor-B: Identification of Amino Acids Important for Receptor Binding. J Mol Biol 359: 76.

50. Stroud RM, Wells JA (2004) Mechanistic Diversity of Cytokine Receptor Signaling Across Cell Membranes. Sci STKE 231: re7.

51. Greenwald J, Groppe J, Gray P, Wiater E, Kwiatkowski W, Vale W, Choe S (2003) The BMP7/ActRII Extracellular Domain Complex Provides New Insights into the Cooperative Nature of Receptor AAssembly. Mol Cell 11:605.

52. Gront D, Kmiecik S, Kolinski A (2007) Backbone building from quadrilaterals. A fast and accurate algorithm for protein backbone reconstruction from alpha carbon coordinates. J Comput Chemistry 28: 1593-1597.

53.Szymczak P, Cieplak M (2006) Stretching of proteins in a uniform flow.J Chem Phys 125:164903.

54. Evans E, Ritchie K (1999) Strength of Weak Bond Connecting Flexible Polymer Chains. Biophys J 76: 2439-2447.

55. Seifert U (2000) Rupture of multiple parallelmolecular binds under dynamic loading. Phys Rev Lett 84: 2750-2753.

56. Carrion-Vazquez M, Li HB, Lu H, Marszalek PE, Oberhauser AF, et al. (2003) The mechanical stability of ubiquitin is linkage dependent. Nat Struct Biol 10: 738-743.

57. Chyan CL, Lin FC, Peng H, Yuan JM, Chang CH, et al. (2003) Reversible mechanical unfolding of single ubiquitin molecules. Biophys J 87: 3995-4006.

58. Dietz H, Berkemeier F, Bertz M, Rief M (2006) Anisotropic deformation response of single protein molecules. Proc Natl Acad Sci U S A 103: 12724-12728.

59.Watanabe K, Muhle-Goll C, Kellermayer MSZ, Labeit S, Granzier HL(2002) Different molecular mechanics displayed by titin's constitutively and differen-tially expressed tandem Ig segment. Struct Biol J 137: 248-258.

60. Watanabe K, Nair P, Labeit D, Kellermayer MSZ, Greaser M, et al. (2002) Molecular mechanics of cardiac titins PEVK and N2B spring elements. J Biol Chem 277:11549-11558.

61. Li HB,Fernandez JM (2003) Mechanical design of the first proximal Ig domain of hman cardiac titin revealed by single molecule force spectroscopy.J Mol Biol 334:75-86.

62.Yang G, Cecconi C, Baase WA, Vetter IR, Breyer WA, et al. (2000) Solid-state synthesis and mechanical unfolding of polymers of T4 lysozyme. Proc Nat Acad Sci U S A 97: 139-144.

63. Lenne PF, Raae AJ, Altmann SM, Saraste M, Horber JKH (2000) States and transition dluring unfolding of a single spectrin repeat. FEBS Lett 476: 124-128.

64.Carrion-Vazquez M, Oberhauser AF, Fisher TE, Marszalek PE, Li HB, et al. (2000)Mechanical design of proteins studied by single-molecule force. Prog Biophys Mol Biol 74: 63-91.

65. Best RB, Li B, Steward A, Daggett V, Clarke J (2001) Can non-mechanical proteins withstand force? Stretching barnase by atomic force microscopy and molecular dynamics simulation. Biophys J 81: 2344-2356.

66. Brockwell DJ, Godfrey S, Beddard S, Paci E, West DK, et al. (2005) Mechanically unfolding small topologically simple protein L. Biophys J 89: 506-519.

67. Dietz H, Rief M (2004) Exploring the energy landscape of the GFP by single-molecule mechanical experiments. Proc Natl Acad Sci U S A 101: 16192-16197.

68. Dietz H, Rief M (2006) Protein structure by mechanical triangulation. Proc Natl Acad Sci U S A 103:1244-1247.

69. Li L, Han-Li Huang H, Badilla CL, Fernandez JM (2005) Mechanical unfolding intermediates observed by single-molecule spectroscopy in fibronectin type III module. J Mol Biol 345: 817-826.

70. Oberhauser AF, Badilla-Fernandez C, Carrion-Vazquez M, Fernandez JM (2002) The mechanical hierarchies of fibronectin observed with single molecule AFM.J Mol Biol 319:433-447.

71. Oberdorfer Y, Fuchs H, Janshoff A (2000) Conformational analysis of native fibronectin by means of force spectroscopy. Langmuir 16: 9955-9958.

72. Oberhauser AF, Marszalek PE, Erickson HP, Fernandez JM (1998) The molecular elasticity of the extracellular matrix protein tenascin. Nature 14: 181-185.

73. Cao Y, Li HB (2007) Polyprotein of GB1 is an ideal artificial elastomeric protein. Nature Mat 6: 109-114.

<!-- PLoS Computational Biology | www.ploscompbiol.org 15 October 2009 | Volume 5 | Issue 10 |e1000547 -->

<!--  -->

Copyright of PLoS Computational Biology is the property of Public Library of Science and its content may not be copied or emailed to multiple sites or posted to a listserv without the copyright holder's express written permission. However, users may print, download, or email articles for individual use.

